<!DOCTYPE HTML>
<html>
<head>
  <link rel="stylesheet" type="text/css" href="styles.css">

  <title>Kevin Lin, UPenn Statistics</title>
	<meta name="google-site-verification" content="_icP2W3B6y_gFG92Dh-caUWIh_sEnE4tPu8pghzaH2w" />
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
  <link href="https://fonts.googleapis.com/css?family=Julius+Sans+One" rel="stylesheet">
  <link rel="stylesheet" href="css/style.css">
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
	<script>
	$(document).ready(function(){
	  $("a").on('click', function(event) {
	    if (this.hash !== "") {
	      event.preventDefault();

	      var hash = this.hash;

	      $('html, body').animate({
	        scrollTop: $(hash).offset().top
	      }, 500, function(){
	        window.location.hash = hash;
	      });
	    }
	  });
	});
	</script>
</head>

<body>
  <div class="banner">
    <div id="header">
      <div id="name">
        Kevin Lin
      </div>
      <div id="navigator">
        <div class="nav_cell"><a class="nav" href="#about">about</a></div>
        <div class="nav_cell"><a class="nav" href="#research">research</a></div>
        <div class="nav_cell"><a class="nav" href="#talks">talks</a></div>
        <div class="nav_cell"><a class="nav" href="#experience">experience</a></div>
        <div class="nav_cell"><a class="link" href="resources.html">resources</a></div>
      </div>
    </div>
  </div>

  <div id="main">

    <div class="content">
      <a class="anchor" id="about"></a>
      <div class="introduction">
        <div id="selfie">
          <img src="images/selfie.jpg" /><br>
          <div style="text-align: center; margin-top: 10px; font-size: 12px; font-family: monospace;">kzlin [at] uw.edu</div>
        </div>
        <div class="right">
        <p>I am a tenure-track assistant professor at <a class="link" href="https://www.biostat.washington.edu/" target="_blank">UW Biostatistics</a>, having started in Fall 2023.
        Previously, I received my Ph.D. from the
        <a class="link" href="http://www.stat.cmu.edu/" target="_blank">Department of Statistics & Data Science</a> at <a class="link" href="https://www.cmu.edu/" target="_blank">Carnegie Mellon University</a>, under supervision of Dr. <a class="link" href="" target="_blank" data-link="kathryn_roeder">Kathryn Roeder</a> and Dr. <a class="link" href="" target="_blank" data-link="jing_lei">Jing Lei</a>, and completed my post-doctoral training
         with Dr. <a class="link" href="" target="_blank" data-link="nancy_zhang">Nancy Zhang</a>.
        I am excited to be joining  as an tenure-track assistant professor starting August 2023.
        </p>

        <p>My research mainly focuses on investigating regulation
        among biological systems (such as in the developing brain to
        understand the autism spectrum or in immune cells to understand
        acquired resistance to immunotherapy) through
        single-cell data.
        This entails
        developing statistical methods, and leveraging recent
        statistical-theoretical developments
        in areas such as matrix factorizations and
        network analysis.
        Recently, I have been focused on developing these methods
        tailored for paired multiomic data where we can directly
        observe the relationships between two modalities.
        The code for my research are available on <a class="link" href="https://github.com/linnykos" target="_blank">git</a>, and my Google Scholar page is <a class="link" href="https://scholar.google.com/citations?user=0ARIIHYAAAAJ&hl=en" target="_blank">here</a>.
        </p>

<!--        <p>
        I also have a M.S. in <a class="link" href="https://www.ml.cmu.edu/" target="_blank">Machine Learning</a> from
        <a class="link" href="https://www.cmu.edu/" target="_blank">Carnegie Mellon University</a>.
		    Before coming to CMU, I got my B.S.E. in <a class="link" href="https://www.orfe.princeton.edu/" target="_blank">Operations Research and Financial Engineering</a> from <a class="link" href="https://www.princeton.edu/" target="_blank">Princeton University</a> in 2014, along with a Certificate in Statistics and Machine Learning as well as a Certificate in Applications of Computing.
        </p>
-->

        <p>
        When taking a break from work, I am a fanatic fan of zumba, cooking, Yu-Gi-Oh, and poetry. Follow my adventures on <a class="link" href="https://www.instagram.com/linnykos/?hl=en" target="_blank">Instagram</a>! (Fun fact: The "Z." in "Kevin Z. Lin" is simply
        for publishing purposes, and is not an official middle name.)
        </p>

      </div>
    </div>
    <hr>

    <div class="content">
      <div class="title">Research</div>
      <a class="anchor" id="research"></a>

<div class="subtitle">Preprints/In Preparation</div>


<!-- dynamic SBM -->
<div class="block">
  <div class="abstract">
    <img class="project_img" src="images/plot-dynamicsbm.png">
    <div class="right">
      <div class="research_title">
       Dynamic stochastic block models
      </div>
      <p>
    In genomics, we often observe a sequence of graphs over some notion of time, such as the gene network
    ordered by the cells' developmental age. However, there is a lack of concrete
    statistical theory for this setting whose assumptions are amendable for single-cell analyses.
    In this work, we develop an estimator designed for models where the underlying stochastic block model
    is smoothly changing with time, and prove the convergence rates of the clustering and estimates of the connectivity
    matrix under assumptions more general than those existing in the literature. We apply this to
    study the dynamics of gene co-expression networks among oligodendrocytes.
      </p>
    </div>
  </div>
  <div class="publication">
    <span class="emph">K. Z. Lin</span> and
                    <a class="link" href="http://www.stat.cmu.edu/~jinglei/" target="_blank">J. Lei</a><br>
                   Spectral clustering for heterophilic stochastic block models with time-varying node memberships<br>
                   (<a class="link" href="papers/dynamicSBM.pdf" target="_blank">main text</a>)
  </div>
</div>

<!-- eSVD-de -->
<div class="block">
  <div class="abstract">
    <img class="project_img" src="images/plot-esvd-de.png">
    <div class="right">
      <div class="research_title">
       eSVD for differential expression in multi-individual scRNA data
      </div>
      <p>
    As single-cell RNA-seq (scRNA) transitions from studying cell-lines, clonal mice or
    a few individuals into studying cohorts of individuals, there is a need to
    create new differential expression (DE) methods to investigate DE among individuals
    instead of among cells. Towards this end, we tailor the eSVD, formerly a dimension-reduction tool, equipped with a downstream
    pipeline to perform DE in multi-individual scRNA data, where we both design our test statistics to account for variability among and within individuals as well as incorporating
    the individual-level covariates into the eSVD framework. We show that this can recover
    reproducible signals across different datasets as well as achieves higher power thanks
    to its dimension-reduction framework.
      </p>
    </div>
  </div>
  <div class="publication">
    <span class="emph">K. Z. Lin</span>, <a class="link" href="https://statr.me/" target="_blank">Y. Qiu</a>, and <a class="link" href="https://kathrynmroeder.github.io/" target="_blank">K. Roeder</a><br>
                    eSVD: Cohort-level differential expression in single-cell RNA-seq data using exponential-family embeddings<br>
 (<a class="link" href="papers/cohort_eSVD.pdf" target="_blank">main text</a>)
  </div>
</div>



<!-- dependency -->
<div class="block">
      <div class="abstract">
        <img class="project_img" src="images/plot-dependency.png">
        <div class="right">
          <div class="research_title">
            Dependency diagnostic via scatter plots
          </div>
          <p>
Dependency graphs encode complex pairwise patterns that are often statistically estimated, but are often hard to diagnose with visualizations due to the quadratic number of scatter plots.
In this paper, we develop an interactive system in R that learns if the data scientist visually interprets dependency.
Then, this system applies the learned classifier to infer a dependency graph that can be compared against the estimated graph. This paper won honorable mention for the Student Paper Award in the ASA Section: <a class="link" href="http://stat-computing.org/awards/student/winners.html" target="_blank">Statistical Computing and Statistical Graphics</a> (2018).
         </p>
        </div>
      </div>
      <div class="publication">
        <span class="emph">K. Z. Lin</span> and T. Kim
<!--		  <a class="link" href="https://www.linkedin.com/in/amy-tian-797892153" target="_blank">A. Tian</a>
		  and
		  <a class="link" href="http://magics.cs.northwestern.edu/index.html" target="_blank">H. Liu</a> -->
		  <br>
        Interactive dependency visualizer: Understanding pairwise variable relationships, applied to single-cell RNA-seq data<br>
		 In preparation
      </div>
    </div>



<div class="subtitle">Published papers</div>

   <!-- maxfuse -->
<div class="block">
      <div class="abstract">
        <img class="project_img" src="images/plot-maxfuse.png">
        <div class="right">
          <div class="research_title">
            MaxFuse: Diagonal integration of single-cell sequencing data
          </div>
          <p>
Different sequencing technologies offer complementary insights into a biological
system, but it becomes difficult to integrate two datasets together when both
datasets differ in the specific cells sequenced as well as which features are measured.
Hence, using an iterative co-embedding based on CCA, data smoothing, and cell matching, we match cells in one dataset (i.e., one modality) to the other.
The effectiveness of our method is demonstrated by integrating scRNA-seq
and surface-antibody data, as well as scRNA-seq data with spatial proteomic data (such as CODEX) and scATAC-seq.
         </p>
        </div>
      </div>
      <div class="publication">
        <a class="link" href="https://sxc.moe/" target="_blank">S. Chen</a>,
        B. Zhu,
        S. Huang,
        J. W. Hickey,
        <span class="emph">K. Z. Lin</span>,
        <a class="link" href="https://med.stanford.edu/snyderlab.html" target="_blank">M. Snyder</a>,
        <a class="link" href="https://greenleaf.stanford.edu/" target="_blank">W. J. Greenleaf</a>,
        <a class="link" href="http://web.stanford.edu/group/nolan/" target="_blank">G. P. Nolan</a>,
        <a class="link" href="https://nzhanglab.github.io/" target="_blank">N. R. Zhang</a>, and
        <a class="link" href="https://zmastat.github.io/" target="_blank">Z. Ma</a>
		  <br>
        Integration of spatial and single-cell data across modalities with weak linkage<br>
      <a class="link" href="https://www.nature.com/nbt/" target="_blank">Nature Biotechnology</a> (To appear, 2023). (<a class="link" href="https://www.biorxiv.org/content/10.1101/2023.01.12.523851v2" target="_blank">biorxiv</a>)
      </div>
    </div>

<!-- tilted-cca -->
<div class="block">
  <div class="abstract">
    <img class="project_img" src="images/plot-tcca.png">
    <div class="right">
      <div class="research_title">
       Tilted-CCA for paired multiomic single-cell data
      </div>
      <p>
    Paired multiomic single-cell data provides ample opportunities for biologists
    to study the relations between molecular modalities (such as gene expression and chromatin
    peaks), but it difficult to assess what kind of signals are shared in common
    between the two modalities. We developed the Tilted-CCA, which shows the "intersection
    of information" between the two modalities, in contrast with many existing
    dimension-reduction methods that show the "union of information." In particular,
    the matrix decomposition provided by Tilted-CCA allows us to design targeted
    antibody panels for RNA and surface-antibody technologies, as well as uncover
    the terminal cell-states as well as the relation between chromatin open/closed cell-states
    and gene expression in developmental systems.
      </p>
    </div>
  </div>
  <div class="publication">
    <span class="emph">K. Z. Lin</span>, and <a class="link" href="https://nzhanglab.github.io/" target="_blank">N. R. Zhang</a><br>
                    Quantifying common and distinct information in single-cell multimodal data with Tilted-CCA<br>
                   <a class="link" href="https://www.pnas.org/" target="_blank">Proceedings of the National Academy of Sciences</a> (PNAS) (To appear, 2023). (<a class="link" href="https://www.pnas.org/doi/10.1073/pnas.2303647120" target="_blank">link</a>) (<a class="link" href="https://www.biorxiv.org/content/10.1101/2022.10.07.511320v1" target="_blank">biorxiv</a>)
  </div>
</div>


      <!-- destin2 -->
<div class="block">
   <div class="abstract">
     <img class="project_img" src="images/plot-destin2.png">
     <div class="right">
       <div class="research_title">
         Destin2: Unifying peaks, motifs and activity in single-cell ATAC-seq
       </div>
       <p> There are different signals a researcher can extract from single-cell ATAC-seq, and each has their own biological relevance for downstream analysis.
       However, one might want to perform computational tasks that aggregate
       all such signals together.
       We propose a computational framework to aggregate across such signals
       for downstream visualization, cell-clustering and trajectory inference.
       </p>
     </div>
   </div>
   <div class="publication">
      P. Y. Guan,
      JS. Lee,
      L. Wang,
      <span class="emph">K. Z. Lin</span>,
      W. Mei,
      <a class="link" href="https://biostat.ufl.edu/profile/chen-li-1/" target="_blank">L. Chen</a>, and
      <a class="link" href="https://yuchaojiang.github.io/" target="_blank">Y. Jiang</a>.
                     <br>
                     Destin2: Integrative and cross-modality analysis of single-cell chromatin accessibility data<br>
                     <i><a class="link" href="https://www.frontiersin.org/journals/genetics" target="_blank">Frontiers in Genetics</a></i> 14 (2023), (<a class="link" href="https://www.frontiersin.org/articles/10.3389/fgene.2023.1089936/full" target="_blank">link</a>) (<a class="link" href="https://www.biorxiv.org/content/10.1101/2022.11.04.515202v1" target="_blank">biorxiv</a>)
   </div>
 </div>


      <!-- sosd -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-sosd.png">
          <div class="right">
            <div class="research_title">
             Bias-adjusted spectral embedding for multi-layer networks
            </div>
            <p>
           While clustering of nodes is now theoretically well-understood for
           a single network through the lense of spectral embeddings,
           the theoretical understanding for clustering in multi-layer networks
           is less well-understood. In this paper, when the clustering
           structure is shared across all the layers, we develop a simple
           way to aggregate information across layers
           via squaring the adjacency matrix with an appropriate
           bias-adjustment. Our theoretical analysis shows we can both allow for
           networks with dissociative structure (i.e., negative eigenvalues) and
           provably obtains consistent clustering even if each individual
           network is extremely sparse. We apply our method to
           study cluster of genes in developing monkey brains.
            </p>
          </div>
        </div>
        <div class="publication">
          <a class="link" href="http://www.stat.cmu.edu/~jinglei/" target="_blank">J. Lei</a> and
          <span class="emph">K. Z. Lin</span><br>
                          Bias-adjusted spectral clustering in multi-layer
                          stochastic block models<br>
                         <i> <a class="link" href="https://amstat.tandfonline.com/toc/uasa20/current" target="_blank">Journal of the American Statistical Association</a> (JASA)</i>
                         (To appear, 2022)
                         (<a class="link" href="https://amstat.tandfonline.com/doi/full/10.1080/01621459.2022.2054817" target="_blank">link</a>)
                         (<a class="link" href="https://arxiv.org/abs/2003.08222" target="_blank">arxiv</a>)
                         (<a class="link" href="https://github.com/linnykos/networkSoSD" target="_blank">git</a>)
        </div>
      </div>

<!-- cpreg -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-cpreg.png">
          <div class="right">
            <div class="research_title">
              Changepoint detection in high-dimensional regression
            </div>
            <p>
            Methods to detect multiple changepoints in regression coefficients typically have
            localization rate (i.e., rate to estimate the location of the changepoints) that scale inversely with the square root with number of
            samples. In this paper, we develop VPWBS,
            a method that appropriately transforms the regression problem into a one-dimensional problem upon which we apply WBS. We prove the convergence rate
            scales inversely with number of samples, a substantial improvement over existing rates.
            </p>
          </div>
        </div>
        <div class="publication">
          <a class="link" href="https://darenwang.github.io/website/" target="_blank">D. Wang</a>,
           <a class="link" href="https://sites.google.com/view/zifengzhao/" target="_blank">Z. Zhao</a>,
                          <span class="emph">K. Z. Lin</span>, and
                          <a class="link" href="https://voices.uchicago.edu/willett/" target="_blank">R. Willet</a><br>
                          Statistically and computationally efficient change point
localization in regression settings<br>
                         <a class="link" href="https://www.jmlr.org/" target="_blank">Journal of Machine Learning Research</a> (JMLR) 22.248 (2021): 1-46 (<a class="link" href="https://jmlr.org/papers/v22/19-531.html" target="_blank">link</a>) (<a class="link" href="https://arxiv.org/abs/1906.11364" target="_blank">arxiv</a>)
        </div>
      </div>


<!-- singlecell -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-singlecell.png">
          <div class="right">
            <div class="research_title">
             Exponential-family embedding for cell developmental trajectories
            </div>
            <p>
           Common pipelines to estimate the cell developmental trajectories
           based on single-cell data typically first embed each cell into a lower-dimensional space, but these embedding typically assume statistical models
           that do not model single-cell data well. In this paper, we develop
           an embedding for hierarchical model where the inner product between
           two latent low-dimensional vectors is the natural parameter of an
           exponential family distributed random variable, and prove identifiability and convergence. When studying oligodendrocytes in fetal mouse brains, we find
           that oligodendrocytes mature into various cell types.
            </p>
          </div>
        </div>
        <div class="publication">
          <span class="emph">K. Z. Lin</span>,
                          <a class="link" href="http://www.stat.cmu.edu/~jinglei/" target="_blank">J. Lei</a>, and
                          <a class="link" href="https://kathrynmroeder.github.io/" target="_blank">K. Roeder</a><br>
                          Exponential-family embedding with application to cell developmental trajectories for single-cell RNA-seq data<br>
                         <i> <a class="link" href="https://amstat.tandfonline.com/toc/uasa20/current" target="_blank">Journal of the American Statistical Association</a> (JASA)</i>  116.534 (2021): 457-470
                         (<a class="link" href="https://www.tandfonline.com/doi/abs/10.1080/01621459.2021.1886106" target="_blank">link</a>)
                          (<a class="link" href="https://biorxiv.org/cgi/content/short/2020.09.25.313882v1" target="_blank">biorxiv</a>)
                         (<a class="link" href="https://github.com/linnykos/esvd" target="_blank">git</a>)
        </div>
      </div>

<!-- binseg -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-binseg.png">
          <div class="right">
            <div class="research_title">
              Post-selection inference for changepoint problems
            </div>
            <p>
            Changepoint detection methods such as binary segmentation are often used in CGH analyses for copy number variation detection, but these methods lack proper downstream statistical inference. In this paper, we develop
            post-selection hypothesis tests for various
            changepoint detection methods, prove our sampling strategies' validity, and provide substantial practical guidelines based on simulation.
            </p>
          </div>
        </div>
        <div class="publication">
          <a class="link" href="http://sangwon-hyun.org/" target="_blank">S. Hyun</a>,
                          <span class="emph">K. Z. Lin</span>,
                          <a class="link" href="http://www.stat.cmu.edu/people/faculty/mgsell" target="_blank">M. G'Sell</a>, and
                          <a class="link" href="http://www.stat.cmu.edu/~ryantibs/index.html" target="_blank"> R. J. Tibshirani</a><br>
                          Valid post-selection inference for segmentation methods with application to copy number variation data<br>
                         <i> <a class="link" href="https://onlinelibrary.wiley.com/journal/15410420" target="_blank">Biometrics</a></i> 77.3 (2021): 1037-1049
                          (<a class="link" href="https://onlinelibrary.wiley.com/doi/10.1111/biom.13422" target="_blank">link</a>)
 (<a class="link" href="https://arxiv.org/abs/1812.03644" target="_blank">arxiv</a>)
  (<a class="link" href="papers/binseginf.pdf" target="_blank">pdf</a>)
 (<a class="link" href="https://github.com/robohyun66/binseginf" target="_blank">git 1</a>, <a class="link" href="https://github.com/linnykos/selectiveModel" target="_blank">git 2</a>)
        </div>
      </div>


<!-- brainspan -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-brainspan.png">
          <div class="right">
            <div class="research_title">
              Covariance-based sample selection for heterogeneous data
            </div>
            <p> Microarray samples from brain tissue is hard to collect, and also varies substantially depending on the tissue's brain region and the developmental age of its subject, hence it is hard to collect enough samples for the statistical analysis.
            In this paper, we develop a sample selection method to find additional microarray samples that are statistically similar to the samples of our desired spatio-temporal brain tissue.
            We demonstrate that after apply an existing analysis pipeline to our selected samples, we detect a higher percentage of autism risk genes.
            </p>
          </div>
        </div>
        <div class="publication">
          <span class="emph">K. Z. Lin</span>,
                          <a class="link" href="http://magics.cs.northwestern.edu/index.html" target="_blank">H. Liu</a>, and
                          <a class="link" href="https://kathrynmroeder.github.io/" target="_blank">K. Roeder</a> <br>
                          Covariance-based sample selection for heterogeneous data: Applications to gene expression and autism risk gene detection<br>
                         <i> <a class="link" href="https://amstat.tandfonline.com/toc/uasa20/current" target="_blank">Journal of the American Statistical Association</a> (JASA)</i> 116.533 (2021): 54-67
                         (<a class="link" href="https://www.tandfonline.com/doi/abs/10.1080/01621459.2020.1738234?journalCode=uasa20" target="_blank">link</a>)
(<a class="link" href="https://arxiv.org/abs/1812.08147" target="_blank">arxiv</a>)
(<a class="link" href="https://github.com/linnykos/covarianceSelection" target="_blank">git</a>)
        </div>
      </div>

<!-- ecv -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-ecv.png">
          <div class="right">
            <div class="research_title">
             Discussion paper: Network cross validation (Li et. al, 2020)
            </div>
            <p>
            We discuss <a class="link" href="https://arxiv.org/abs/1612.04717" target="_blank">"Network cross-validation by edge sampling"</a> by Li, Levina, and Zhu (Biometrika, 2020)
            where empirically explore the possibilities of combining the ideas of the authors with
            ideas providing confidence sets for cross-validated parameters as well as extending
            the ideas of the authors into the network tensor setting.
            </p>
          </div>
        </div>
        <div class="publication">
          <a class="link" href="http://www.stat.cmu.edu/~jinglei/" target="_blank">J. Lei</a>,
          <span class="emph">K. Z. Lin</span><br>
                          Discussion of 'Network cross-validation by edge sampling'<br>
                         <i> <a class="link" href="https://academic.oup.com/biomet" target="_blank">Biometrika</a></i> 107.2 (2020): 285-287.
 (<a class="link" href="https://academic.oup.com/biomet/article-abstract/107/2/285/5837741" target="_blank">link</a>)
 (<a class="link" href="papers/ecv_disc.pdf" target="_blank">pdf</a>)
 (<a class="link" href="https://github.com/linnykos/edgeCV" target="_blank">git</a>)
        </div>
      </div>


<div class="block">
   <div class="abstract">
     <img class="project_img" src="images/plot-science.png">
     <div class="right">
       <div class="research_title">
         Autism risk factors within the noncoding genome
       </div>
       <p> While de novo mutations within the protein-coding portion
       of the genome have
       been thoroughly studied, these mutations in the noncoding portions
       which comprise of 98.5% of the genome
       have been less well understood. In this paper, we use a bioinformatics
       framework that builds upon (among other things)
       sparse PCA and DAWN based on simulated null datasets to analyze 1902 autism
       quartets via WGS and find that
       the
       strongest signals arose from promoters -- noncoding regions that
       control gene transcription.
       </p>
     </div>
   </div>
   <div class="publication">
      <a class="link" href="https://sanderslab.github.io/team/joonan.html" target="_blank">J. An*</a>,
      <span class="emph">K. Z. Lin</span>*,
      <a class="link" href="https://lingxuez.github.io/" target="_blank">L. Zhu*</a>,
      <a class="link" href="https://sanderslab.github.io/team/donnawerling" target="_blank">D. M. Werling*</a>,
      S. Dong, H. Brand, H. Z. Wang, X. Zhao, G. B. Schwartz, R. L. Collins,
      B. B. Currall, C. Dastmalchi, J. Dea, C. Duhn, M. C. Gilson, L. Klei,
      L. Liang, E. Markenscoff-Papadimitriou, S. Pochareddy, N. Ahituv,
      J. D. Buxbaum, H. Coon, M. J. Daly, Y. S. Kim, G. T. Marth,
      B. M. Neale, A. R. Quinlan, J. L. Rubenstein, N. Sestan,
      M. W. State, A. J. Willsey,
      <a class="link" href="http://talkowski.mgh.harvard.edu/about/people/" target="_blank">M. E. Talkowski</a>,
      <a class="link" href="https://www.psychiatry.pitt.edu/about-us/our-people/faculty/bernie-j-devlin-phd" target="_blank">B. Devlin</a>,
      <a class="link" href="https://kathrynmroeder.github.io/" target="_blank">K. Roeder</a>,
      <a class="link" href="https://sanderslab.github.io/team/stephansanders" target="_blank">S. J. Sanders</a>. (*Equal contributions)
                     <br>
                     Genome-wide de novo risk score implicates promoter variation in autism spectrum disorder<br>
                     <i><a class="link" href="http://science.sciencemag.org/" target="_blank">Science</a></i> 362.6420 (2018).
                     (<a class="link" href="https://www.science.org/doi/full/10.1126/science.aat6576" target="_blank">link</a>)
                 (<a class="link" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6432922/pdf/nihms-1015808.pdf" target="_blank">pdf</a>) (<a class="link" href="https://github.com/lingxuez/WGS-Analysis" target="_blank">git</a>)
   </div>
 </div>

<!-- fused lasso -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-fused.png">
          <div class="right">
            <div class="research_title">
              Fused lasso and changepoint screening
            </div>
            <p> Changepoint estimators have statistical theory for how well they estimate the mean function and how well they estimate the changepoints, but existing theory often analyzes these properties separately. In this paper, we prove a near-optimal estimation rate for the fused lasso, which in turn directly proves a changepoint detection rate that is near the detection limit. We extend this logic to other estimators and settings.
            </p>
          </div>
        </div>
        <div class="publication">
          <span class="emph">K. Z. Lin</span>,
                          <a class="link" href="https://jsharpna.github.io/" target="_blank">J. Sharpnack</a>,
                          <a class="link" href="http://www.stat.cmu.edu/~arinaldo/" target="_blank">A. Rinaldo</a>, and
                          <a class="link" href="http://www.stat.cmu.edu/~ryantibs/index.html" target="_blank">R. J. Tibshirani</a> <br>
                          A sharp error analysis for the fused Lasso, with application to approximate changepoint screening<br>
                          <i><a class="link" href="https://nips.cc/" target="_blank">Advances in Neural Information Processing Systems</a> (NeurIPS)</i> (2017).
                          (<a class="link" href="http://papers.nips.cc/paper/7264-a-sharp-error-analysis-for-the-fused-lasso-with-application-to-approximate-changepoint-screening" target="_blank">link</a>)
                      (<a class="link" href="https://arxiv.org/abs/1606.06746" target="_blank">arxiv</a>) (<a class="link" href="https://github.com/linnykos/fused_lasso" target="_blank">git</a>)
        </div>
      </div>



<div class="subtitle">One-off Projects</div>



<div class="block">
   <div class="abstract">
     <img class="project_img" src="images/plot-wiki.png">
     <div class="right">
       <div class="research_title">
         Social biases in Wikipedia biographies
       </div>
       <p> Social biases can influence Wikipedia articles of individuals
       across different genders and ethnicities that result in disproportiate
       article lengths, number of languages written about said individuals, etc.,
       but these differences can be difficult to quantify appropriately.
       In this paper, we develop a matching method to find an appropriate
       comparison set of articles
       for each target group of interest (ex: articles of African American individuals) based on the articles' categories. We show the differing covariate distributions across the different target groups, and uncover quantitative results that reinforce existing social theories. This paper won the <a class="link" href="https://research.wikimedia.org/awards.html" target="_blank">Wikimedia Foundation Research Award of the Year</a> in 2023.
       </p>
     </div>
   </div>
   <div class="publication">
      <a class="link" href="https://www.cs.cmu.edu/~anjalief/" target="_blank">A. Field</a>,
        <a class="link" href="https://chan0park.github.io/" target="_blank">C. Y. Park</a>,
      <span class="emph">K. Z. Lin</span>,
      <a class="link" href="https://homes.cs.washington.edu/~yuliats/" target="_blank">Y. Tsvetkov</a>.
                     <br>
                     Controlled Analyses of Social Biases in Wikipedia Bios<br>
                     <i><a class="link" href="https://www2022.thewebconf.org/" target="_blank">Proceedings of the ACM Web Conference 2022</a></i>, (<a class="link" href="https://dl.acm.org/doi/abs/10.1145/3485447.3512134" target="_blank">link</a>) (<a class="link" href="https://arxiv.org/abs/2101.00078" target="_blank">arxiv</a>) (<a class="link" href="https://github.com/anjalief/wikipedia_bias_public" target="_blank">git</a>)
   </div>
 </div>

<!-- cs -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-cs.png">
          <div class="right">
            <div class="research_title">
              Compressed sensing
            </div>
            <p> Many compressed sensing are developed to be as generic as possible, but have shortcomings in specialized settings where modern optimization theory can deliver a substantial boost in computational efficiency. In this paper, we develop two compressed sensing algorithms, one specialized for extremely sparse signals and another specialized for Kronecker-structed sensing matrices. We
            numerically demonstrate a near 10-times reduction in
            computation time compared to other state-of-the-art methods.
            </p>
          </div>
        </div>
        <div class="publication">
          <a class="link" href="https://vanderbei.princeton.edu/" target="_blank">R. Vanderbei</a>,
          <span class="emph">K. Z. Lin</span>,
                          <a class="link" href="http://magics.cs.northwestern.edu/index.html" target="_blank">H. Liu</a>, and
                          <a class="link" href="http://math.mit.edu/~liewang/" target="_blank">L. Wang</a><br>
                          Revisiting compressed sensing: Exploiting the efficiency
of simplex and sparsification methods<br>
                          <i><a class="link" href="https://www.springer.com/mathematics/journal/12532" target="_blank">Mathematical Programming Computation</a></i> 8.3 (2016): 253-269.
                          (<a class="link" href="https://link.springer.com/article/10.1007/s12532-016-0105-y" target="_blank">link</a>)
                      (<a class="link" href="papers/cs.pdf" target="_blank">pdf</a>)
                      (<a class="link" href="https://github.com/linnykos/compress_sensing" target="_blank">git</a>)
        </div>
      </div>


<div class="content">
  <div class="title">Articles</div>
  <a class="anchor" id="articles"></a>

  <div class="block">
    <div class="abstract">
        <div class="application_title">
      We, the millenials: The statistical significance of political significance
        </div>
     <div class="publication">
         <i><a class="link" href="https://www.significancemagazine.com/" target="_blank">Significance</a></i> (2017)
         (<a class="link" href="https://www.significancemagazine.com/politics/574-we-the-millennials-the-statistical-significance-of-political-significance" target="_blank">link</a>)
         </div>
      </div>
</div>

    <hr>

    <div class="content">
      <div class="title">Talks and posters</div>
      <a class="anchor" id="talks"></a>

      <ul>

          <li>
			  <i>Spectral clustering for heterophilic stochastic block models with dynamic node memberships</i> (talk).<br>
			  2023 International Chinese Statistical Association (ICSA) Applied Statistics Symposium, Ann Arbor, MI.
			  </li>

           <li>
			  <i>Tilted-CCA: Quantifying Common and Distinct Information in Jointly-Sequenced Multiomic Single-Cell Data</i> (talk).<br>
			  2022 Joint Statistical Meetings (JSM), Washington DC.
			  </li>

           <li>
			  <i>Spectral clustering for multi-layer stochastic block models: Theoretical analysis of static and dynamic settings for heterophilic networks</i> (talk).<br>
			  2022 Symposium on Data Science and Statistics (SDSS), Pittsburgh, PA.
			  </li>

         <li>
			  <i>Exponential-family embedding for single-cell data with applications to developmental trajectory and differential expression</i> (talk).<br>
			  2021 UCLA Department of Statistics: Seminar Series. (Remote)
			  </li>

        <li>
			  <i>Time-varying stochastic block models, with application to understanding the dynamics of gene co-expression</i> (talk).<br>
			  2021 StatScale Seminar. (Remote)
			  </li>

        <li>
			  <i>Exponential-family embedding with application to cell developmental trajectories for single-cell RNA-Seq data</i> (poster).<br>
			  2020 American Society of Human Genetics (ASHG). (Remote)
			  </li>

       		  <li>
			  <i>Exponential-family embedding with application to cell developmental trajectories for single-cell RNA-Seq data</i> (talk).<br>
			  Joint talk with Professor <a class="link" href="https://kathrynmroeder.github.io/" target="_blank">Kathryn Roeder</a><br>
			 2020 Joint Statistical Meetings (JSM), Philadelphia, Pennslyvania. (Remote)
			  </li>

			  		  <li>
			  <i>Time-varying stochastic block models via kernel smoothing, with application to RNA-Seq data and cell development</i> (talk).<br>
			  2020 Joint Statistical Meetings (JSM), Philadelphia, Pennslyvania. (Remote)
			  </li>

			  		  <li>
			  <i>Exponential-family embedding with application to cell developmental trajectories for single-cell RNA-Seq data</i> (talk).<br>
			  2019 Joint Statistical Meetings (JSM), Denver, Colorado.
			  </li>

			  <li>
			  <i>Dependency diagnostic: Visually understanding pairwise variable relationships</i> (talk).<br>
			  2018 Joint Statistical Meetings (JSM), Vancouver, Canada.
			  </li>

			  <li>
			  <i>A sharp error analysis for the fused lasso, with application to approximate changepoint screening</i> (poster).<br>
			  2017 Conference on Neural Information Processing Systems (NeurIPS), Long Beach, CA.
			  </li>

			  <li>
			  <i>Hypothesis testing for simultaenous variable clustering and correlation network estimation, with applications to gene coexpression networks</i> (talk).<br>
			  2017 Joint Statistical Meetings (JSM), Baltimore, MD.
			  </li>

			   <li>
			  <i>Longitudinal Gaussian graphical model for autism risk gene detection</i> (talk).<br>
			  2016 Joint Statistical Meetings (JSM), Chicago, IL.
			  </li>

			  <li>
			  <i>Longitudinal Gaussian graphical model integrating gene expression and sequencing data for autism risk gene detection</i> (poster).<br>
			  2015 American Society of Human Genetics (ASHG), Baltimore, MD.
			  </li>

			  <li>
			  <i>Optimization for compressed sensing: New insights and alternatives</i> (talk).<br>
			  2014 Modeling and Optimization: Theory and Applications, Bethlehem, PA.
			  </li>
      </ul>
    </div>

    <hr>

    <div class="content">
      <div class="title">Experience</div>
      <a class="anchor" id="experience"></a>

      <div class="subtitle">Honors and Awards</div>
      <ul>

          <li>
        <i>PhD TAs of the year</i><br>
        (For the Spring 2020 semester, one of two recipients)<br>
        Carnegie Mellon University, May 2020
        </li>
             <li>
        <i>Honorable mention in student paper competition</i><br>
        (For article "Dependency diagnostic: Visually understanding pairwise variable relationships")<br>
        ASA section: Statistical Computing and Statistical Graphics, January 2018
        </li>

        <li>
        <i>Winner of Statistical excellence for early-career writing</i><br>
        (For article "We, the millennials: The statistical significance of political significance")<br>
        Significance magazine in partnership with Young Statisticians Section of Royal Statistical Society, June 2017
        </li>

        <li>
        <i>Teaching assistant award recipient</i><br>
        (For "Statistical Computing" in Fall 2016)<br>
        Carnegie Mellon University, May 2017
        </li>

         <li>
        <i>Award recipient of Kenneth H. Condit Prize</i><br>
        (For excellence in service to department)<br>
        Princeton University, May 2014
        </li>

      </ul>

    <div class="subtitle">Teaching Experience</div>

      <ul>
        <li>(2022 January) Section on Statistics in Genetics and Genomics: An Introduction to Deep Learning in Omics (TA for a virtual short course under
        <a class="link" href="https://research.fredhutch.org/sun/en.html" target="_blank">W. Sun</a>
        and
        <a class="link" href="https://nzhanglab.github.io/" target="_blank">N. R. Zhang</a>)</li>
       <li>(2020 Spring) CMU: 36-469 Statistical Genomics and High Dimensional Inference (co-instructor with <a class="link" href="https://kathrynmroeder.github.io/" target="_blank">K. Roeder</a>)</li>
        <li>(2019 Spring) CMU: 36-490 Undergraduate Research (Data Science Initiative Project Fellow under <a class="link" href="http://www.stat.cmu.edu/~rnugent/PUBLIC/" target="_blank">R. Nugent</a> and <a class="link" href="http://www.stat.cmu.edu/~pfreeman/" target="_blank">P. Freeman</a>)</li>
			   <li>(2018 Summer) CMU: 36-350 Statistical Computing (Instructor)</li>
			   <li>(2018 Spring) CMU: 36-350 Statistical Computing (Assistant Instructor with <a class="link" href="http://www.stat.cmu.edu/~ryantibs/index.html" target="_blank">R. J. Tibshirani</a>)</li>
			   <li>(2017 Fall) 36-350 CMU: Statistical Computing (TA under <a class="link" href="http://www.stat.cmu.edu/~pfreeman/" target="_blank">P. Freeman</a>)</li>
			   <li>(2015 Fall, 2016 Fall) CMU: 36-350 Statistical Computing (TA under <a class="link" href="http://www.stat.cmu.edu/~ryantibs/index.html" target="_blank">R. J. Tibshirani</a>)</li>
			    <li>(2015 Spring) CMU: 36-217 Probability Theory and Random Processes (TA under <a class="link" href="http://www.stat.cmu.edu/~arinaldo/" target="_blank">A. Rinaldo</a>)</li>
			    <li>(2014 Fall) CMU: 46-921 Financial Data Analysis I and 46-923 Financial Data Analysis II (TA under <a class="link" href="http://www.stat.cmu.edu/~cschafer/" target="_blank">C. Schafer</a>)</li>
			    <li>(2014 Spring, 2013 Spring, 2012 Spring) Princeton: ORF 350 Analysis of Big Data (Course designer with <a class="link" href="http://magics.cs.northwestern.edu/index.html" target="_blank">H. Liu</a>)</li>
      </ul>

      <div class="subtitle">Professional Service</div>

      <ul>
         <li>
          Certified by <a class="link" href="https://www.mentalhealthfirstaid.org/" target="_blank">Mental Health First Aid USA</a> (September 2020)
          </li>
        <li>
          Certified by <a class="link" href="https://https://www.cmu.edu/teaching/graduatestudentsupport/futurefacultyprogram.html.com/" target="_blank">CMU's Eberly Center's Future Faculty Program</a> (Fall 2019 to Summer 2020)
          </li>
        <li>
          Certified via <a class="link" href="https://qprinstitute.com/" target="_blank">QPR (Question, Persuade, Refer)</a> gatekeeper certificate by the QPR suicide prevention gatekeeper program (February 2020)
          </li>
          <li>
          Founding member of CMU Statistics department wellness network (2018-2020)
          </li>
        <li>
          Reviewer of
          <a class="link" href="https://projecteuclid.org/journals/annals-of-applied-statistics" target="_blank">Annals of Applied Statistics</a>,
          <a class="link" href="https://projecteuclid.org/euclid.aos" target="_blank">Annals of Statistics</a>,
          <a class="link" href="https://projecteuclid.org/journals/bayesian-analysis" target="_blank">Bayesian Analysis</a>,
          <a class="link" href="https://onlinelibrary.wiley.com/journal/15410420" target="_blank">Biometrics</a>,
          <a class="link" href="https://academic.oup.com/biomet" target="_blank">Biometrika</a>,
          <a class="link" href="https://projecteuclid.org/euclid.ejs" target="_blank">Electronic Journal of Statistics</a>,
          <a class="link" href="https://ieeexplore.ieee.org/xpl/RecentIssue.jsp?punumber=6488902" target="_blank">IEEE Transactions on Network Science and Engineering</a>,
          <a class="link" href="https://www.jmlr.org/" target="_blank">Journal of Machine Learning Research</a>,
          <a class="link" href="https://www.journals.elsevier.com/journal-of-molecular-biology" target="_blank">Journal of Molecular Biology</a>,
          <a class="link" href="https://www.tandfonline.com/toc/uasa20/current" target="_blank">Journal of the American Statistical Association</a>,
          <a class="link" href="https://www.nature.com/neuro/" target="_blank">Nature Neuroscience</a>,
          <a class="link" href="http://www3.stat.sinica.edu.tw/statistica/" target="_blank">Statistical Sinica</a>,
          <a class="link" href="https://www.journals.elsevier.com/statistics-and-probability-letters" target="_blank">Statistics and Probability Letters</a>,
          <a class="link" href="https://onlinelibrary.wiley.com/journal/10970258" target="_blank">Statistics in Medicine</a>,
          <a class="link" href="https://www.tandfonline.com/toc/utch20/current" target="_blank">Technometrics</a>
          </li>
      </ul>



    </div>
  <div style="margin-top: 10px; text-align: center; font-size: 14px;">
  Copyright &copy; 2015 - 2023, Kevin Lin. All rights reserved.<br>
  Last Updated: July 31, 2023
  </div>

<script src="script.js"></script>
</body>
</html>
