<!DOCTYPE HTML>
<html>
<head>
  <link rel="stylesheet" type="text/css" href="styles.css">

  <title>Kevin Z. Lin, UW Biostatistics</title>
	<meta name="google-site-verification" content="_icP2W3B6y_gFG92Dh-caUWIh_sEnE4tPu8pghzaH2w" />
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
  <link href="https://fonts.googleapis.com/css?family=Julius+Sans+One" rel="stylesheet">
  <link rel="stylesheet" href="css/style.css">
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
	<script>
	$(document).ready(function(){
	  $("a").on('click', function(event) {
	    if (this.hash !== "") {
	      event.preventDefault();

	      var hash = this.hash;

	      $('html, body').animate({
	        scrollTop: $(hash).offset().top
	      }, 500, function(){
	        window.location.hash = hash;
	      });
	    }
	  });
	});
	</script>
</head>

<body>
  <div class="banner">
    <div id="header">
      <div id="name">
        Kevin Z. Lin
      </div>
      <div id="navigator">
        <div class="nav_cell"><a class="nav" href="#about">about</a></div>
        <div class="nav_cell"><a class="nav" href="#research">research</a></div>
        <div class="nav_cell"><a href="people.html">people</a></div>
        <div class="nav_cell"><a href="resources.html">resources</a></div>
        <div class="nav_cell"><a href="papers/cv.pdf">cv</a></div>
      </div>
    </div>
  </div>

  <div id="main">

    <div class="content">
      <a class="anchor" id="about"></a>
      <div class="introduction">
        <div id="selfie">
          <img src="images/selfie.png" /><br>
          <div style="text-align: center; margin-top: 10px; font-size: 12px; font-family: monospace;">kzlin [at] uw.edu</div>
          <div style="text-align: center; margin-top: 10px; font-size: 12px; font-family: monospace;">(206) 685-8334</div>
          <div style="text-align: center; margin-top: 10px; font-size: 12px; font-family: monospace;">Office: 338 HRC</div>
          <div style="text-align: center; margin-top: 10px; font-size: 12px; font-family: monospace;"><a href="https://scholar.google.com/citations?user=0ARIIHYAAAAJ&hl=en" target="_blank">Google Scholar</a></div>
        </div>
        <div class="right">
        <p>I am a Genentech Endowed tenure-track assistant professor at <a href="https://www.biostat.washington.edu/" target="_blank">UW Biostatistics</a>, and started in Fall 2023.
        Previously, I received my Ph.D. from the
        <a href="http://www.stat.cmu.edu/" target="_blank">Department of Statistics & Data Science</a> at <a  href="https://www.cmu.edu/" target="_blank">Carnegie Mellon University</a>, under supervision of Dr. <a class="link" href="" target="_blank" data-link="kathryn_roeder">Kathryn Roeder</a> and Dr. <a class="link" href="" target="_blank" data-link="jing_lei">Jing Lei</a>, and completed my post-doctoral training
         with Dr. <a class="link" href="" target="_blank" data-link="nancy_zhang">Nancy R. Zhang</a>.
        </p>

        <p>My research focuses on human cellular mechanisms in Alzheimer’s 
        disease (AD) using single-cell data from post-mortem tissue 
        (sequencing and imaging).
        Statistical methods offer powerful clues, but these current methods
        leave many fundamental questions unanswered.
        • (1) How can we use post-mortem tissue to infer deteriorating cellular 
        mechanisms ante-mortem? 
        We can detect differences between tissue from AD donors and 
        cognitively/pathologically normal donors, but do these differences 
        reflect long-term dynamics that persisted for decades before death? 
        • (2) Brain cells respond not only to amyloid-β and neurofibrillary tau 
        tangles (the defining lesions of AD) but these proteins spread 
        differently across the brain, often influenced by co-occurring 
        neurodegenerative diseases. 
        How can we isolate AD-specific cellular responses amid substantial 
        person-to-person variation, region-specific progression, aging, and 
        unmeasured co-pathologies? 
        • (3) Some donors show high AD pathological burden post-mortem yet 
        no ante-mortem cognitive decline. 
        How can we rigorously quantify 
        this "resilience," and what protective mechanisms allow these donors 
        to mitigate the neurotoxicity of amyloid-β and tau? 
        To address all these different questions about AD,
        my lab develops statistical and computational advances (ex: deep learning, 
        matrix factorization, and assumption-lean hypothesis testing) to 
        build the next generation of dry-lab tools to inform future therapeutic strategies.
        </p>
        
        <p>
        <span class="emph">Funding</span>: It is my honor to be funded by the 
        <a href="https://www.washington.edu/research/or/royalty-research-fund-rrf/" target="_blank">UW Royalty Research Fund</a> (2024-25) as a pilot grant to support the lab.
        </p>
        
        <p>
        <span class="emph">Course notes</span>: With help from student volunteers, I also maintain online <a href ="https://linnykos.github.io/scOmicNotes/" target = "_blank">lecture notes</a>
        that stemmed from "BIOST 545: Biostatistical Methods for Big Omics Data" (Winter 2025), 
        which provides an overview of how statistical/computational methods specialize to
        understand different 'omics from single-cell sequencing data.
        </p>

        <p>
        When taking a break from work, I am a fanatic fan of zumba and cooking. Follow my adventures on <a href="https://www.instagram.com/linnykos/?hl=en" target="_blank">Instagram</a>! (Fun fact: The "Z." in "Kevin Z. Lin" is simply
        for publishing purposes, and is not an official middle name.)
        </p>

        <p>
        <b>Note</b>: There are multiple "Kevin Lin"'s at the University of Washington. Be sure to contact me at the correct email (ID: kzlin)!
        </p>


      </div>
    </div>
    <hr>

    <div class="content">
      <div class="title">Research</div>
      <a class="anchor" id="research"></a>

<div class="subtitle">Preprints/In Preparation</div>
(Bold denotes members of or closely associated with the Lin Lab.)

<!-- GeoAdvAE -->
<div class="block">
  <div class="abstract">
  <img class="project_img" src="images/plot-geoadvae.png">
  <div class="right">
  <div class="research_title">
  GeoAdvAE: Diagonal integration of neural cell's morphology and transcriptomics
  </div>
  <p>
 A cell's morphology is a powerful readout of a cell's function and state.
 While microscopy allows us to study neural cells' morphology, and single-nuclei RNA-sequencing allows us to study a cell's transcriptomics,
 these two rich data sources are often performed on separate cells due to technological limitations.
 This obstructs us from understanding how a neural cell's transcriptomics modulates or responds to changes in its morphology.
 We develop a deep-learning diagonal integration method, to overcome this technological limitation.
 We validate our method on Patch-seq neurons and study genes that control microglial morphology from the 5xFAD mouse model for Alzheimer's disease.
  </p>
  </div>
  </div>
  <div class="publication">
    <span class="emph"><a class="link" href="" target="_blank" data-link="turbo_du">J. T. Du</a></span>,
    <span class="emph">K. Z. Lin</span>.
  <br>
  Integrating morphology and gene expression of neural cells in unpaired single-cell data using GeoAdvAE<br>
  2025: (<a href="https://www.biorxiv.org/content/10.1101/2025.11.19.689368v1" target="_blank">biorxiv</a>)
  (<a href="https://github.com/turbodu222/GeoAdVAE" target="_blank">code: method and analysis</a>)
  </div>
</div>

<!-- Clonotrace -->
<div class="block">
  <div class="abstract">
  <img class="project_img" src="images/plot-clonotrace.png">
  <div class="right">
  <div class="research_title">
  Clonotrace: Fate bias and pseudotime from joint modeling of single-cell lineage barcode and transcriptomics
  </div>
  <p>
 Single-cell technologies now measure both gene expression and clonal ancestry, 
 but most analysis tools still treat these modalities separately. 
 We develop a framework that clusters clones by their "clonotype profiles"
 (the distributions of their cells across transcriptomic space) to jointly model lineage and 
 expression and recover cell-state transitions and fate biases. 
 Applied to simulated and experimental systems including hematopoiesis, cancer 
 drug resistance models, and CD8 T cells from glioblastoma and immunotherapy cohorts, 
 our method reconstructs temporal order across abrupt transcriptional shifts, 
 distinguishes unipotent from multipotent progenitors, and uncovers subtle, 
 recurrent fate-divergent T cell programs. 
  </p>
  </div>
  </div>
  <div class="publication">
    <a class="link" href="" target="_blank" data-link="yuntian_fu">Y. Fu</a>,
    <a class="link" href="" target="_blank" data-link="divij_mathew">D. Mathew</a>,
    M. Wang,
    <a class="link" href="" target="_blank" data-link="emilia_chen">X. E. Chen</a>,
    <span class="emph">K. Z. Lin</span>,
    <a class="link" href="" target="_blank" data-link="sydney_shaffer">S. Shaffer</a>,
    <a class="link" href="" target="_blank" data-link="drew_pardoll">D. M. Pardoll</a>,
    <a class="link" href="" target="_blank" data-link="christina_jackson">C. Jackson</a>,
    <a class="link" href="" target="_blank" data-link="nancy_zhang">N. R. Zhang</a>.
  <br>
  Deciphering Cell Fate and Clonal Dynamics via Integrative Single-Cell Lineage Modeling<br>
  2025: (<a href="https://www.biorxiv.org/content/10.1101/2025.09.01.673503v1" target="_blank">biorxiv</a>)
  (<a href="https://github.com/yuntianf/Clonotrace" target="_blank">code: method</a>)
  </div>
</div>

<!-- ePRS -->
<div class="block">
  <div class="abstract">
  <img class="project_img" src="images/plot-eprs.png">
  <div class="right">
  <div class="research_title">
  Enndolysosomal dysfunction across neural cells in AD
  </div>
  <p>
 It is unclear whether how genetic variants linked to Alzheimer’s disease (AD) like the endolysosomal network (ELN) cause functional disruptions in the brain.
 We developed a pathway-specific polygenic risk score (ePRS) based on 13 ELN-related AD GWAS loci and analyzed single-nucleus transcriptomes and endolysosomal morphology in post-mortem DLPFC samples from AD patients stratified by ePRS burden.
 Using confocal imaging and snRNA-seq, we identified cell-type-specific morphological and transcriptional signatures associated with higher ELN genetic risk, including altered endosome and lysosome structures, dysregulation of glutamatergic signaling, and increased markers of cellular stress.
 These findings demonstrate that genetic risk in ELN genes contributes to distinct and shared cellular phenotypes across neural cell types, suggesting that pathway-specific risk variants may drive mechanistic heterogeneity in AD.
  </p>
  </div>
  </div>
  <div class="publication">
    <a class="link" href="" target="_blank" data-link="sainath_mamde">S. Mamde</a>,
    S. E. Rose,
    <a class="link" href="" target="_blank" data-link="katherine_prater">K. E. Prater</a>,
    A. Cochoit,
    Y. F. Lin,
    <a class="link" href="" target="_blank" data-link="isa_smith">I. Smith</a>,
    <a class="link" href="" target="_blank" data-link="corbin_johnson">C. S. Johnson</a>,
    <a class="link" href="" target="_blank" data-link="aquene_reid">A. N. Reid</a>,
    W. Qiu,
    S. Strohbehn,
    <a class="link" href="" target="_blank" data-link="dirk_keene">C. D. Keene</a>,
    <a class="link" href="" target="_blank" data-link="brad_rolf">B. A. Rolf</a>,
    <span class="emph">K. Z. Lin</span>,
    <a class="link" href="" target="_blank" data-link="su_in_lee">S. I. Lee</a>,
    <a class="link" href="" target="_blank" data-link="gwenn_garden">G. A. Garden</a>,
    <a class="link" href="" target="_blank" data-link="elizabeth_blue">E. E. Blue</a>,
    <a class="link" href="" target="_blank" data-link="jessica_young">J. E. Young</a>, and
     <a class="link" href="" target="_blank" data-link="suman_jayadev">S. Jayadev</a>.
  <br>
  Genetic risk in endolysosomal network genes correlates with endolysosomal dysfunction across neural cell types in Alzheimer's disease<br>
  2025: (<a href="https://www.biorxiv.org/content/10.1101/2025.03.16.643481v3" target="_blank">biorxiv</a>)
  </div>
</div>

<!-- ODE -->
<div class="block">
  <div class="abstract">
  <img class="project_img" src="images/plot-ode.png">
  <div class="right">
  <div class="research_title">
  ODEinherit: Mother-daughter inheritance in cell-cycling
  </div>
  <p>
  Cells can undergo asymmetric cell division, and in certain systems, the fate of the daughter cell can be predicted even before it buds from the mother cell. However, it is not well understood what exactly is inherited by the daughter cell. We study budding yeast cells via live-cell microscopy to study the protein-protein regulatory network of mother cells and their daughter cells as time series, and develop a framework based on ODE regulatory networks to estimate the the precentage a daughter cell inherits its protein-protein regulatory network from its mother at single-cell resolution.
  </p>
  </div>
  </div>
  <div class="publication">
    <span class="emph">W. Wu</span>,
    T. Kennedy,
    <a class="link" href="" target="_blank" data-link="orlando_miranda">O. Arguello-Miranda</a>, and <span class="emph">K. Z. Lin</span>.
  <br>
  Measuring regulatory network inheritance in dividing yeast cells using ordinary differential equations<br>
  2024: (<a href="https://www.biorxiv.org/content/10.1101/2024.11.23.624995v2" target="_blank">biorxiv</a>)
  (<a href="https://github.com/WenbinWu2001/ODEinherit" target="_blank">code: method</a>)  (<a href="https://wenbinwu2001.github.io/ODEinherit/" target="_blank">code: tutorial</a>)
  </div>
</div>

<!-- Replicate -->
<div class="block">
  <div class="abstract">
  <img class="project_img" src="images/plot-replicate.png">
  <div class="right">
  <div class="research_title">
 Replicat coherence: Replicate coherence in RNA velocity via count splitting
  </div>
  <p>
  RNA velocity has been a transformative tool to study development biology via single-cell RNA sequencing. However, although many RNA velocity methods exist, it is common for many methods to conflict in the biological interpretation of the results. We develop a method called replicate coherence to quantify the uncertainty in the RNA velocity vectors where we use the count-splitting framework to emulate having two technical replicates of the same exact cells. This allows us to quantify how much uncertainty each RNA velocity result has, which enables a more accurate side-by-side comparison of different methods for the scRNA-seq dataset at hand.
  </p>
  </div>
  </div>
  <div class="publication">
    <span class="emph">Y. Li</span>,
    Z. J. Wei,
    <a class="link" href="" target="_blank" data-link="yenchi_chen">Y.-C. Chen</a>, and <span class="emph">K. Z. Lin</span>.
  <br>
  Assessing RNA velocity stability across synthetic
replicates using count splitting<br>
  2024: (<a href="https://www.biorxiv.org/content/10.1101/2024.11.23.625009v2" target="_blank">biorxiv</a>)
  (<a href="https://github.com/linnykos/veloUncertainty" target="_blank">code: method and analysis</a>)
  </div>
</div>


<!-- LCL -->
<div class="block">
  <div class="abstract">
  <img class="project_img" src="images/plot-lcl.png">
  <div class="right">
  <div class="research_title">
  LCL: Lineage-aware contrastive learning
  </div>
  <p>
  In static lineage-barcoding single-cell sequencing, we measure both each cell's gene expression and its ancestry in order to gain insights into a cell's fate. However, there lack methods to identify which genes separate the cells by their lineages. In this work, we develop a deep-learning architecture based on contrastive learning to isolate the information in the gene expression that recapitulates cells' lineages. We use the lineage barcoe as the premise of our data augmentation mechanism. This strategy enables us to better identify the genes driving fate commitment and cells' fate boundaries.
  </p>
  </div>
  </div>
  <div class="publication">
   <span class="emph"><a class="link" href="" target="_blank" data-link="joshua_yang">S. J. Yang</a></span>,
    <a class="link" href="" target="_blank" data-link="yixin_wangi">Y. Wang</a>, and <span class="emph">K. Z. Lin</span>.
  <br>
  LCL: Contrastive Learning for Lineage Barcoded scRNA-seq Data<br>
  2024: (<a href="https://www.biorxiv.org/content/10.1101/2024.10.28.620670v1" target="_blank">biorxiv</a>)
  (<a href="https://github.com/SZ-yang/Lineage-aware_CL" target="_blank">code: method and analysis</a>)
  </div>
</div>

<!-- dynamic SBM -->
<div class="block">
  <div class="abstract">
  <img class="project_img" src="images/plot-dynamicsbm.png">
  <div class="right">
  <div class="research_title">
  KD-SoS: Dynamic stochastic block models
  </div>
  <p>
  In genomics, we often observe a sequence of graphs over some notion of time, such as the gene network
  ordered by the cells' developmental age. However, there is a lack of concrete
  statistical theory for this setting whose assumptions are amendable for single-cell analyses.
  In this work, we develop an estimator designed for models where the underlying stochastic block model
  is smoothly changing with time, and prove the convergence rates of the clustering and estimates of the connectivity
  matrix under assumptions more general than those existing in the literature. We apply this to
  study the dynamics of gene co-expression networks among oligodendrocytes.
  </p>
  </div>
  </div>
  <div class="publication">
  <span class="emph">K. Z. Lin</span>  and
  <a class="link" href="" target="_blank" data-link="jing_lei">J. Lei</a>.
  <br>
  Dynamic clustering for heterophilic stochastic block models with time-varying node memberships<br>
  2024: (<a href="https://arxiv.org/abs/2403.05654" target="_blank">arxiv</a>)  (<a href="https://github.com/linnykos/dynamicGraphRoot" target="_blank">code: method and analysis</a>)
  </div>
</div>




<div class="subtitle">Published papers</div>

<!-- Glia -->
<div class="block">
  <div class="abstract">
  <img class="project_img" src="images/plot-glia.png">
  <div class="right">
  <div class="research_title">
  Single-cell analysis review in glial cells
  </div>
  <p>
  Computational single-cell analyses have become quite complex.
  While there are many tools for any particular specific analysis, researchers new to single-cell analyses might often feel overwhelmed and not immediately understand the logical flow that connects one computational method to another.
  This review paper highlights the underlying logic of how to craft a computational workflow, tailored for the biological context of studying glial cells.
  It starts with thinking about the experimental design, and spotlights common pitfalls when planning your own computational workflow.
  </p>
  </div>
  </div>
  <div class="publication">
  <a class="link" href="" target="_blank" data-link="katherine_prater">K. E. Prater</a> and <span class="emph">K. Z. Lin</span>.
  <br>
  All the single cells: Single-cell transcriptomics/epigenomics experimental design and analysis considerations for glial biologists<br>
  <a href="https://onlinelibrary.wiley.com/journal/10981136" target="_blank">Glia</a>
     73 (2025):
     (<a href="https://onlinelibrary.wiley.com/doi/10.1002/glia.24633" target="_blank">link</a>)
     (<a href="https://arxiv.org/abs/2408.06521" target="_blank">arxiv</a>)
  </div>
</div>


<!-- eSVD-de -->
<div class="block">
  <div class="abstract">
    <img class="project_img" src="images/plot-esvd-de.png">
    <div class="right">
      <div class="research_title">
       eSVD-DE: Differential expression in multi-individual scRNA data
      </div>
      <p>
    As single-cell RNA-seq (scRNA) transitions from studying cell-lines, clonal mice or
    a few individuals into studying cohorts of individuals, there is a need to
    create new differential expression (DE) methods to investigate DE among individuals
    instead of among cells. Towards this end, we tailor the eSVD, formerly a dimension-reduction tool, equipped with a downstream
    pipeline to perform DE in multi-individual scRNA data, where we both design our test statistics to account for variability among and within individuals as well as incorporating
    the individual-level covariates into the eSVD-DE framework. We show that this can recover
    reproducible signals across different datasets as well as achieves higher power thanks
    to its dimension-reduction framework.
      </p>
    </div>
  </div>
  <div class="publication">
    <span class="emph">K. Z. Lin</span>,
    <a class="link" href="" target="_blank" data-link="yixuan_qiu">Y. Qiu</a>, and
    <a class="link" href="" target="_blank" data-link="kathryn_roeder">K. Roeder</a>.
    <br>
                    eSVD-DE: Cohort-wide differential expression in single-cell RNA-seq data using exponential-family embeddings<br>
                    <a href="https://bmcbioinformatics.biomedcentral.com/" target="_blank">BMC Bioinformatics</a>
      25.1 (2024):
      (<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-024-05724-7" target="_blank">link</a>)
                   (<a href="https://www.biorxiv.org/content/10.1101/2023.11.22.568369v2" target="_blank">biorxiv</a>)
                   (<a href="https://github.com/linnykos/eSVD2" target="_blank">code: method</a>)
                   (<a href="https://github.com/linnykos/eSVD2_examples" target="_blank">code: analysis</a>)
                   (<a href="https://linnykos.github.io/eSVD2/index.html" target="_blank">code: tutorials</a>)
  </div>
</div>

   <!-- maxfuse -->
<div class="block">
      <div class="abstract">
        <img class="project_img" src="images/plot-maxfuse.png">
        <div class="right">
          <div class="research_title">
            MaxFuse: Diagonal integration of single-cell sequencing data
          </div>
          <p>
Different sequencing technologies offer complementary insights into a biological
system, but it becomes difficult to integrate two datasets together when both
datasets differ in the specific cells sequenced as well as which features are measured.
Hence, using an iterative co-embedding based on CCA, data smoothing, and cell matching, we match cells in one dataset (i.e., one modality) to the other.
The effectiveness of our method is demonstrated by integrating scRNA-seq
and surface-antibody data, as well as scRNA-seq data with spatial proteomic data (such as CODEX) and scATAC-seq.
         </p>
        </div>
      </div>
      <div class="publication">
        <a class="link" href="" target="_blank" data-link="shuxiao_chen">S. Chen</a>,
        B. Zhu,
        S. Huang,
        J. W. Hickey,
        <span class="emph">K. Z. Lin</span>,
        <a class="link" href="" target="_blank" data-link="shuxiao_chen">M. Snyder</a>,
        <a class="link" href="" target="_blank" data-link="willaim_greenleaf">W. J. Greenleaf</a>,
        <a class="link" href="" target="_blank" data-link="garry_nolan">G. P. Nolan</a>,
        <a class="link" href="" target="_blank" data-link="nancy_zhang">N. R. Zhang</a>, and
        <a class="link" href="" target="_blank" data-link="zongming_ma">Z. Ma</a>.
		  <br>
        Integration of spatial and single-cell data across modalities with weak linkage<br>
      <a href="https://www.nature.com/nbt/" target="_blank">Nature Biotechnology</a>
 42 (2023):
      (<a href="https://www.nature.com/articles/s41587-023-01935-0" target="_blank">link</a>)
      (<a href="https://www.biorxiv.org/content/10.1101/2023.01.12.523851v2" target="_blank">biorxiv</a>)
      (<a href="https://github.com/shuxiaoc/maxfuse" target="_blank">code: method</a>)
      (<a href="https://maxfuse.readthedocs.io/en/latest/" target="_blank">code: tutorials</a>)
      </div>
    </div>

<!-- tilted-cca -->
<div class="block">
  <div class="abstract">
    <img class="project_img" src="images/plot-tcca.png">
    <div class="right">
      <div class="research_title">
       Tilted-CCA: Disentangling common/distinct information from paired multiomic single-cell data
      </div>
      <p>
    Paired multiomic single-cell data provides ample opportunities for biologists
    to study the relations between molecular modalities (such as gene expression and chromatin
    peaks), but it difficult to assess what kind of signals are shared in common
    between the two modalities. We developed the Tilted-CCA, which shows the "intersection
    of information" between the two modalities, in contrast with many existing
    dimension-reduction methods that show the "union of information." In particular,
    the matrix decomposition provided by Tilted-CCA allows us to design targeted
    antibody panels for RNA and surface-antibody technologies, as well as uncover
    the terminal cell-states as well as the relation between chromatin open/closed cell-states
    and gene expression in developmental systems.
      </p>
    </div>
  </div>
  <div class="publication">
    <span class="emph">K. Z. Lin</span>, and
     <a class="link" href="" target="_blank" data-link="nancy_zhang">N. R. Zhang</a>.
    <br>
                    Quantifying common and distinct information in single-cell multimodal data with Tilted-CCA<br>
                   <a href="https://www.pnas.org/" target="_blank">Proceedings of the National Academy of Sciences</a> (PNAS)
                   120.32 (2023):
                   (<a href="https://www.pnas.org/doi/10.1073/pnas.2303647120" target="_blank">link</a>)
                   (<a href="https://www.biorxiv.org/content/10.1101/2022.10.07.511320v1" target="_blank">biorxiv</a>)
                   (<a href="https://github.com/linnykos/tiltedCCA" target="_blank">code: method</a>)
                   (<a href="https://github.com/linnykos/tiltedCCA_analysis" target="_blank">code: analysis</a>)
                   (<a href="https://linnykos.github.io/tiltedCCA/" target="_blank">code: tutorials</a>)
  </div>
</div>


      <!-- destin2 -->
<div class="block">
   <div class="abstract">
     <img class="project_img" src="images/plot-destin2.png">
     <div class="right">
       <div class="research_title">
         Destin2: Unifying peaks, motifs and activity in single-cell ATAC-seq
       </div>
       <p> There are different signals a researcher can extract from single-cell ATAC-seq, and each has their own biological relevance for downstream analysis.
       However, one might want to perform computational tasks that aggregate
       all such signals together.
       We propose a computational framework to aggregate across such signals
       for downstream visualization, cell-clustering and trajectory inference.
       </p>
     </div>
   </div>
   <div class="publication">
      P. Y. Guan,
      JS. Lee,
      L. Wang,
      <span class="emph">K. Z. Lin</span>,
      W. Mei,
      <a class="link" href="" target="_blank" data-link="li_chen">L. Chen</a>, and
      <a class="link" href="" target="_blank" data-link="yuchao_jiang">Y. Jiang</a>.
                     <br>
                     Destin2: Integrative and cross-modality analysis of single-cell chromatin accessibility data<br>
                     <i><a href="https://www.frontiersin.org/journals/genetics" target="_blank">Frontiers in Genetics</a></i> 14 (2023):
                     (<a href="https://www.frontiersin.org/articles/10.3389/fgene.2023.1089936/full" target="_blank">link</a>)
                     (<a href="https://www.biorxiv.org/content/10.1101/2022.11.04.515202v1" target="_blank">biorxiv</a>)
   </div>
 </div>


      <!-- sosd -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-sosd.png">
          <div class="right">
            <div class="research_title">
             NetworkSoSD: Bias-adjusted spectral embedding for multi-layer networks
            </div>
            <p>
           While clustering of nodes is now theoretically well-understood for
           a single network through the lense of spectral embeddings,
           the theoretical understanding for clustering in multi-layer networks
           is less well-understood. In this paper, when the clustering
           structure is shared across all the layers, we develop a simple
           way to aggregate information across layers
           via squaring the adjacency matrix with an appropriate
           bias-adjustment. Our theoretical analysis shows we can both allow for
           networks with dissociative structure (i.e., negative eigenvalues) and
           provably obtains consistent clustering even if each individual
           network is extremely sparse. We apply our method to
           study cluster of genes in developing monkey brains.
            </p>
          </div>
        </div>
        <div class="publication">
          <a class="link" href="" target="_blank" data-link="jing_lei">J. Lei</a> and
          <span class="emph">K. Z. Lin</span>.
          <br>
                          Bias-adjusted spectral clustering in multi-layer
                          stochastic block models<br>
                         <i> <a href="https://amstat.tandfonline.com/toc/uasa20/current" target="_blank">Journal of the American Statistical Association</a> (JASA)</i>
                         118 (2023):
                         (<a href="https://amstat.tandfonline.com/doi/full/10.1080/01621459.2022.2054817" target="_blank">link</a>)
                         (<a href="https://arxiv.org/abs/2003.08222" target="_blank">arxiv</a>)
                         (<a href="https://github.com/linnykos/networkSoSD" target="_blank">git</a>)
        </div>
      </div>

<!-- cpreg -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-cpreg.png">
          <div class="right">
            <div class="research_title">
              VPWBS: Changepoint detection in high-dimensional regression
            </div>
            <p>
            Methods to detect multiple changepoints in regression coefficients typically have
            localization rate (i.e., rate to estimate the location of the changepoints) that scale inversely with the square root with number of
            samples. In this paper, we develop a method that appropriately transforms the regression problem into a one-dimensional problem upon which we apply WBS. We prove the convergence rate
            scales inversely with number of samples, a substantial improvement over existing rates.
            </p>
          </div>
        </div>
        <div class="publication">
          <a class="link" href="" target="_blank" data-link="daren_wang">D. Wang</a>,
          <a class="link" href="" target="_blank" data-link="zifeng_zhao">Z. Zhao</a>,
          <span class="emph">K. Z. Lin</span>, and
          <a class="link" href="" target="_blank" data-link="rebecca_willet">R. Willet</a>.
          <br>
                          Statistically and computationally efficient change point
localization in regression settings<br>
                         <a href="https://www.jmlr.org/" target="_blank">Journal of Machine Learning Research</a> (JMLR) 22.248 (2021): 1-46
                         (<a href="https://jmlr.org/papers/v22/19-531.html" target="_blank">link</a>)
                         (<a href="https://arxiv.org/abs/1906.11364" target="_blank">arxiv</a>)
        </div>
      </div>


<!-- singlecell -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-singlecell.png">
          <div class="right">
            <div class="research_title">
             eSVD: Exponential-family embedding for cell developmental trajectories
            </div>
            <p>
           Common pipelines to estimate the cell developmental trajectories
           based on single-cell data typically first embed each cell into a lower-dimensional space, but these embedding typically assume statistical models
           that do not model single-cell data well. In this paper, we develop
           an embedding for hierarchical model where the inner product between
           two latent low-dimensional vectors is the natural parameter of an
           exponential family distributed random variable, and prove identifiability and convergence. When studying oligodendrocytes in fetal mouse brains, we find
           that oligodendrocytes mature into various cell types.
            </p>
          </div>
        </div>
        <div class="publication">
          <span class="emph">K. Z. Lin</span>,
                          <a class="link" href="" target="_blank" data-link="jing_lei">J. Lei</a>, and
                          <a class="link" href="" target="_blank" data-link="kathryn_roeder">K. Roeder</a>.
                          <br>
                          Exponential-family embedding with application to cell developmental trajectories for single-cell RNA-seq data<br>
                         <i> <a href="https://amstat.tandfonline.com/toc/uasa20/current" target="_blank">Journal of the American Statistical Association</a> (JASA)</i>  116.534 (2021): 457-470
                         (<a href="https://www.tandfonline.com/doi/abs/10.1080/01621459.2021.1886106" target="_blank">link</a>)
                          (<a href="https://biorxiv.org/cgi/content/short/2020.09.25.313882v1" target="_blank">biorxiv</a>)
                         (<a href="https://github.com/linnykos/esvd" target="_blank">git</a>)
        </div>
      </div>

<!-- binseg -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-binseg.png">
          <div class="right">
            <div class="research_title">
              Post-selection inference for changepoint problems
            </div>
            <p>
            Changepoint detection methods such as binary segmentation are often used in CGH analyses for copy number variation detection, but these methods lack proper downstream statistical inference. In this paper, we develop
            post-selection hypothesis tests for various
            changepoint detection methods, prove our sampling strategies' validity, and provide substantial practical guidelines based on simulation.
            </p>
          </div>
        </div>
        <div class="publication">
          <a class="link" href="" target="_blank" data-link="sangwon_hyun">S. Hyun</a>,
          <span class="emph">K. Z. Lin</span>,
          <a class="link" href="" target="_blank" data-link="max_gsell">M. G'Sell</a>, and
          <a class="link" href="" target="_blank" data-link="ryan_tibshirani">R. J. Tibshirani</a>.
                          <br>
                          Valid post-selection inference for segmentation methods with application to copy number variation data<br>
                         <i> <a href="https://onlinelibrary.wiley.com/journal/15410420" target="_blank">Biometrics</a></i> 77.3 (2021): 1037-1049
                          (<a href="https://onlinelibrary.wiley.com/doi/10.1111/biom.13422" target="_blank">link</a>)
                          (<a href="https://arxiv.org/abs/1812.03644" target="_blank">arxiv</a>)
                          (<a href="papers/binseginf.pdf" target="_blank">pdf</a>)
                          (<a href="https://github.com/robohyun66/binseginf" target="_blank">git 1</a>, <a href="https://github.com/linnykos/selectiveModel" target="_blank">git 2</a>)
        </div>
      </div>


<!-- brainspan -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-brainspan.png">
          <div class="right">
            <div class="research_title">
              Covariance-based sample selection for heterogeneous data
            </div>
            <p> Microarray samples from brain tissue is hard to collect, and also varies substantially depending on the tissue's brain region and the developmental age of its subject, hence it is hard to collect enough samples for the statistical analysis.
            In this paper, we develop a sample selection method to find additional microarray samples that are statistically similar to the samples of our desired spatio-temporal brain tissue.
            We demonstrate that after apply an existing analysis pipeline to our selected samples, we detect a higher percentage of autism risk genes.
            </p>
          </div>
        </div>
        <div class="publication">
          <span class="emph">K. Z. Lin</span>,
          <a class="link" href="" target="_blank" data-link="han_liu">H. Liu</a>, and
          <a class="link" href="" target="_blank" data-link="kathryn_roeder">K. Roeder</a>.
          <br>
                          Covariance-based sample selection for heterogeneous data: Applications to gene expression and autism risk gene detection<br>
                         <i> <a href="https://amstat.tandfonline.com/toc/uasa20/current" target="_blank">Journal of the American Statistical Association</a> (JASA)</i> 116.533 (2021): 54-67
                         (<a href="https://www.tandfonline.com/doi/abs/10.1080/01621459.2020.1738234?journalCode=uasa20" target="_blank">link</a>)
                         (<a href="https://arxiv.org/abs/1812.08147" target="_blank">arxiv</a>)
                         (<a href="https://github.com/linnykos/covarianceSelection" target="_blank">git</a>)
        </div>
      </div>

<!-- ecv -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-ecv.png">
          <div class="right">
            <div class="research_title">
             Discussion paper: Network cross validation (Li et. al, 2020)
            </div>
            <p>
            We discuss <a href="https://arxiv.org/abs/1612.04717" target="_blank">"Network cross-validation by edge sampling"</a> by Li, Levina, and Zhu (Biometrika, 2020)
            where empirically explore the possibilities of combining the ideas of the authors with
            ideas providing confidence sets for cross-validated parameters as well as extending
            the ideas of the authors into the network tensor setting.
            </p>
          </div>
        </div>
        <div class="publication">
          <a class="link" href="" target="_blank" data-link="jing_lei">J. Lei</a> and
          <span class="emph">K. Z. Lin</span>.
          <br>
                          Discussion of 'Network cross-validation by edge sampling'<br>
                         <i> <a href="https://academic.oup.com/biomet" target="_blank">Biometrika</a></i> 107.2 (2020): 285-287.
 (<a href="https://academic.oup.com/biomet/article-abstract/107/2/285/5837741" target="_blank">link</a>)
 (<a href="papers/ecv_disc.pdf" target="_blank">pdf</a>)
 (<a href="https://github.com/linnykos/edgeCV" target="_blank">git</a>)
        </div>
      </div>


<div class="block">
   <div class="abstract">
     <img class="project_img" src="images/plot-science.png">
     <div class="right">
       <div class="research_title">
         Autism risk factors within the noncoding genome
       </div>
       <p> While de novo mutations within the protein-coding portion
       of the genome have
       been thoroughly studied, these mutations in the noncoding portions
       which comprise of 98.5% of the genome
       have been less well understood. In this paper, we use a bioinformatics
       framework that builds upon (among other things)
       sparse PCA and DAWN based on simulated null datasets to analyze 1902 autism
       quartets via WGS and find that
       the
       strongest signals arose from promoters -- noncoding regions that
       control gene transcription.
       </p>
     </div>
   </div>
   <div class="publication">
     <a class="link" href="" target="_blank" data-link="joonyong_an">J. An*</a>,
      <span class="emph">K. Z. Lin</span>*,
      <a class="link" href="" target="_blank" data-link="lingxue_zhu">L. Zhu*</a>,
      <a class="link" href="" target="_blank" data-link="donna_werling">D. M. Werling*</a>,
      S. Dong, H. Brand, H. Z. Wang, X. Zhao, G. B. Schwartz, R. L. Collins,
      B. B. Currall, C. Dastmalchi, J. Dea, C. Duhn, M. C. Gilson, L. Klei,
      L. Liang, E. Markenscoff-Papadimitriou, S. Pochareddy, N. Ahituv,
      J. D. Buxbaum, H. Coon, M. J. Daly, Y. S. Kim, G. T. Marth,
      B. M. Neale, A. R. Quinlan, J. L. Rubenstein, N. Sestan,
      M. W. State, A. J. Willsey,
      <a class="link" href="" target="_blank" data-link="michael_talkowski">M. E. Talkowski</a>,
      <a class="link" href="" target="_blank" data-link="bernie_devlin">B. Devlin</a>,
      <a class="link" href="" target="_blank" data-link="kathryn_roeder">K. Roeder</a>,
      <a class="link" href="" target="_blank" data-link="stephan_sanders">S. J. Sanders</a>. (*Equal contributions)
                     <br>
                     Genome-wide de novo risk score implicates promoter variation in autism spectrum disorder<br>
                     <i><a href="http://science.sciencemag.org/" target="_blank">Science</a></i> 362.6420 (2018):
                     (<a href="https://www.science.org/doi/full/10.1126/science.aat6576" target="_blank">link</a>)
                 (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6432922/pdf/nihms-1015808.pdf" target="_blank">pdf</a>)
                 (<a href="https://github.com/lingxuez/WGS-Analysis" target="_blank">git</a>)
   </div>
 </div>

<!-- fused lasso -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-fused.png">
          <div class="right">
            <div class="research_title">
              Fused lasso and changepoint screening
            </div>
            <p> Changepoint estimators have statistical theory for how well they estimate the mean function and how well they estimate the changepoints, but existing theory often analyzes these properties separately. In this paper, we prove a near-optimal estimation rate for the fused lasso, which in turn directly proves a changepoint detection rate that is near the detection limit. We extend this logic to other estimators and settings.
            </p>
          </div>
        </div>
        <div class="publication">
          <span class="emph">K. Z. Lin</span>,
          <a class="link" href="" target="_blank" data-link="james_sharpnack">J. Sharpnack</a>,
          <a class="link" href="" target="_blank" data-link="alessandro_rinaldo">A. Rinaldo</a>, and
          <a class="link" href="" target="_blank" data-link="ryan_tibshirani">R. J. Tibshirani</a>.
          <br>
                          A sharp error analysis for the fused Lasso, with application to approximate changepoint screening<br>
                          <i><a href="https://nips.cc/" target="_blank">Advances in Neural Information Processing Systems</a> (NeurIPS)</i> (2017):
                          (<a href="http://papers.nips.cc/paper/7264-a-sharp-error-analysis-for-the-fused-lasso-with-application-to-approximate-changepoint-screening" target="_blank">link</a>)
                      (<a href="https://arxiv.org/abs/1606.06746" target="_blank">arxiv</a>)
                      (<a href="https://github.com/linnykos/fused_lasso" target="_blank">git</a>)
        </div>
      </div>



<div class="subtitle">One-off Projects</div>



<div class="block">
   <div class="abstract">
     <img class="project_img" src="images/plot-wiki.png">
     <div class="right">
       <div class="research_title">
         Social biases in Wikipedia biographies
       </div>
       <p> Social biases can influence Wikipedia articles of individuals
       across different genders and ethnicities that result in disproportiate
       article lengths, number of languages written about said individuals, etc.,
       but these differences can be difficult to quantify appropriately.
       In this paper, we develop a matching method to find an appropriate
       comparison set of articles
       for each target group of interest (ex: articles of African American individuals) based on the articles' categories. We show the differing covariate distributions across the different target groups, and uncover quantitative results that reinforce existing social theories. This paper won the <a href="https://research.wikimedia.org/awards.html" target="_blank">Wikimedia Foundation Research Award of the Year</a> in 2023.
       </p>
     </div>
   </div>
   <div class="publication">
     <a class="link" href="" target="_blank" data-link="anjalie_field">A. Field</a>,
     <a class="link" href="" target="_blank" data-link="chan_park">C. Y. Park</a>,
      <span class="emph">K. Z. Lin</span>, and
      <a class="link" href="" target="_blank" data-link="yulia_tsvetkov">Y. Tsvetkov</a>.
                     <br>
                     Controlled Analyses of Social Biases in Wikipedia Bios<br>
                     <i><a href="https://www2022.thewebconf.org/" target="_blank">Proceedings of the ACM Web Conference 2022</a></i> (2022):
                     (<a href="https://dl.acm.org/doi/abs/10.1145/3485447.3512134" target="_blank">link</a>)
                     (<a href="https://arxiv.org/abs/2101.00078" target="_blank">arxiv</a>)
                     (<a href="https://github.com/anjalief/wikipedia_bias_public" target="_blank">git</a>)
   </div>
 </div>


<!-- dependency -->
<div class="block">
      <div class="abstract">
        <img class="project_img" src="images/plot-dependency.png">
        <div class="right">
          <div class="research_title">
            Dependency diagnostic via scatter plots
          </div>
          <p>
Dependency graphs encode complex pairwise patterns that are often statistically estimated, but are often hard to diagnose with visualizations due to the quadratic number of scatter plots.
In this paper, we develop an interactive system in R that learns if the data scientist visually interprets dependency.
Then, this system applies the learned classifier to infer a dependency graph that can be compared against the estimated graph. This paper won honorable mention for the Student Paper Award in the ASA Section: <a href="http://stat-computing.org/awards/student/winners.html" target="_blank">Statistical Computing and Statistical Graphics</a> (2018).
         </p>
        </div>
      </div>
      <div class="publication">
        <span class="emph">K. Z. Lin</span> and T. Kim.
		  <br>
        Interactive dependency visualizer: Understanding pairwise variable relationships, applied to single-cell RNA-seq data<br>
      </div>
    </div>

<!-- cs -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-cs.png">
          <div class="right">
            <div class="research_title">
              Compressed sensing
            </div>
            <p> Many compressed sensing are developed to be as generic as possible, but have shortcomings in specialized settings where modern optimization theory can deliver a substantial boost in computational efficiency. In this paper, we develop two compressed sensing algorithms, one specialized for extremely sparse signals and another specialized for Kronecker-structed sensing matrices. We
            numerically demonstrate a near 10-times reduction in
            computation time compared to other state-of-the-art methods.
            </p>
          </div>
        </div>
        <div class="publication">
          <a class="link" href="" target="_blank" data-link="robert_vanderbei">R. Vanderbei</a>,
          <span class="emph">K. Z. Lin</span>,
          <a class="link" href="" target="_blank" data-link="lie_wang">L. Wang</a>, and
          <a class="link" href="" target="_blank" data-link="han_liu">H. Liu</a>.
                          <br>
                          Revisiting compressed sensing: Exploiting the efficiency
of simplex and sparsification methods<br>
                          <i><a href="https://www.springer.com/mathematics/journal/12532" target="_blank">Mathematical Programming Computation</a></i> 8.3 (2016): 253-269.
                          (<a href="https://link.springer.com/article/10.1007/s12532-016-0105-y" target="_blank">link</a>)
                      (<a href="papers/cs.pdf" target="_blank">pdf</a>)
                      (<a href="https://github.com/linnykos/compress_sensing" target="_blank">git</a>)
        </div>
      </div>


<div class="content">
  <div class="title">Articles</div>
  <a class="anchor" id="articles"></a>

  <div class="block">
    <div class="abstract">
        <div class="application_title">
      We, the millenials: The statistical significance of political significance
        </div>
     <div class="publication">
         <i><a href="https://www.significancemagazine.com/" target="_blank">Significance</a></i> (2017)
         (<a href="https://www.significancemagazine.com/politics/574-we-the-millennials-the-statistical-significance-of-political-significance" target="_blank">link</a>)
         </div>
      </div>
</div>

    <hr>

  <div style="margin-top: 10px; text-align: center; font-size: 14px;">
  Copyright &copy; 2015 - 2025, Kevin Lin. All rights reserved.<br>
  Last Updated: September 13, 2025
  </div>

<script src="script.js"></script>
</body>
</html>
