<!DOCTYPE HTML>
<html>
<head>
  <title>Kevin Lin, CMU Statistics</title>
	<meta name="google-site-verification" content="_icP2W3B6y_gFG92Dh-caUWIh_sEnE4tPu8pghzaH2w" />
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
  <link href="https://fonts.googleapis.com/css?family=Julius+Sans+One" rel="stylesheet">
  <link rel="stylesheet" href="css/style.css">
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
	<script>
	$(document).ready(function(){
	  $("a").on('click', function(event) {
	    if (this.hash !== "") {
	      event.preventDefault();

	      var hash = this.hash;

	      $('html, body').animate({
	        scrollTop: $(hash).offset().top
	      }, 500, function(){
	        window.location.hash = hash;
	      });
	    }
	  });
	});
	</script>
</head>

<body>
  <div class="banner">
    <div id="header">
      <div id="name">
        Kevin Lin
      </div>
      <div id="navigator">
        <div class="nav_cell"><a class="nav" href="#about">about</a></div>
        <div class="nav_cell"><a class="nav" href="#research">research</a></div>
        <div class="nav_cell"><a class="nav" href="#articles">articles</a></div>
        <div class="nav_cell"><a class="nav" href="#talks">talks</a></div>
        <div class="nav_cell"><a class="nav" href="#experience">experience</a></div>
      </div>
    </div>
  </div>

  <div id="main">

    <div class="content">
      <a class="anchor" id="about"></a>
      <div class="introduction">
        <div id="selfie">
          <img src="images/selfie.jpg" /><br>
          <div style="text-align: center; margin-top: 10px; font-size: 12px; font-family: monospace;">klin1234 [at] gmail [dot] com</div>
        </div>
        <div class="right">
        <p>I am working on my Ph.D. from the
        <a class="link" href="http://www.stat.cmu.edu/" target="_blank">Department of Statistics & Data Science</a> at <a class="link" href="https://www.cmu.edu/" target="_blank">Carnegie Mellon University</a>, under supervision of Professor <a class="link" href="http://www.stat.cmu.edu/~roeder/" target="_blank">Kathryn Roeder</a> and  Professor <a class="link" href="http://www.stat.cmu.edu/~jinglei/" target="_blank">Jing Lei</a>.
        My research mainly focuses on developing statistical methods to understand complex data dependencies found in epigenetics applications. Recently, I have been researching changepoint detection and network models. I also have a growing interest in topics relating
        statistics and graph theory.
        The code for my research are available on <a class="link" href="https://github.com/linnylin92" target="_blank">git</a>.
        </p>

        <p>
        I also have a M.S. in <a class="link" href="https://www.ml.cmu.edu/" target="_blank">Machine Learning</a> from
        <a class="link" href="https://www.cmu.edu/" target="_blank">Carnegie Mellon University</a>.
		    Before coming to CMU, I got my B.S.E. in <a class="link" href="https://www.orfe.princeton.edu/" target="_blank">Operations Research and Financial Enginnering</a> from <a class="link" href="https://www.princeton.edu/" target="_blank">Princeton University</a> in 2014, along with a Certificate in Statistics and Machine Learning as well as a Certificate in Applications of Computing.
        </p>

        <p>
        When taking a break from work, I am a fanatic fan of zumba, cooking, anime and poetry.
        </p>

      </div>
    </div>
    <hr>

    <div class="content">
      <div class="title">Research</div>
      <a class="anchor" id="research"></a>

      <div class="subtitle">Preprints/In Preparation</div>

<!-- singlecell -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-singlecell.png">
          <div class="right">
            <div class="research_title">
             Exponential-family embedding for cell developmental trajectories
            </div>
            <p>
           Common pipelines to estimate the cell developmental trajectories
           based on single-cell data typically first embed each cell into a lower-dimensional space, but these embedding typically assume statistical models
           that do not model single-cell data well. In this paper, we develop
           an embedding for hierarchical model where the inner product between
           two latent low-dimensional vectors is the natural parameter of an
           exponential family distributed random variable, and prove identifiability and convergence. When studying oligodendrocytes in fetal mouse brains, we find
           that oligodendrocytes mature into various cell types.
            </p>
          </div>
        </div>
        <div class="publication">
          <span class="emph">K. Lin</span>,
                          <a class="link" href="http://www.stat.cmu.edu/~jinglei/" target="_blank">J. Lei</a>, and
                          <a class="link" href="http://www.stat.cmu.edu/~roeder/" target="_blank">K. Roeder</a><br>
                          Exponential-family embedding with application to cell developmental trajectories for single-cell RNA-seq data<br>
                         In preparation
                         (<a class="link" href="slides/singlecell.pdf" target="_blank">slides</a>) (<a class="link" href="https://github.com/linnylin92/singlecell" target="_blank">git</a>)
        </div>
      </div>

<!-- dependency -->
<div class="block">
      <div class="abstract">
        <img class="project_img" src="images/plot-dependency.png">
        <div class="right">
          <div class="research_title">
            Dependency diagnostic via scatter plots
          </div>
          <p>
Dependency graphs encode complex pairwise patterns that are often statistically estimated, but are often hard to diagnose with visualizations due to the quadratic number of scatter plots.
In this paper, we develop an interactive system in R that learns if the data scientist visually interprets dependency.
Then, this system applies the learned classifier to infer a dependency graph that can be compared against the estimated graph. This paper won honorable mention for the Student Paper Award in the ASA Section: <a class="link" href="http://stat-computing.org/awards/student/winners.html" target="_blank">Statistical Computing and Statistical Graphics</a>.
         </p>
        </div>
      </div>
      <div class="publication">
        <span class="emph">K. Lin</span>,
		  <a class="link" href="https://www.linkedin.com/in/amy-tian-797892153" target="_blank">A. Tian</a>
		  and
		  <a class="link" href="http://magics.cs.northwestern.edu/index.html" target="_blank">H. Liu</a><br>
        Dependency diagnostic: Visually understanding pairwise variable relationships<br>
		 In preparation (<a class="link" href="slides/dependency.pdf" target="_blank">slides</a>) (<a class="link" href="https://github.com/linnylin92/graphicalModelEDA" target="_blank">git</a>)
      </div>
    </div>


<!-- cpreg -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-cpreg.png">
          <div class="right">
            <div class="research_title">
              Changepoint detection in high-dimensional regression
            </div>
            <p>
            Methods to detect changepoints in regression coefficients typically have
            localization rate (i.e., rate to estimate the location of the changepoints) that scale inversely with the square root with number of
            samples. In this paper, we develop a method where we prove the convergence rate
            scales inversely with number of samples, a substantial improvement,
            where we first analyze the convergence towards regression
            coefficients themselves and then apply wild binary segmentation to screen
            for the changepoints.
            </p>
          </div>
        </div>
        <div class="publication">
          <a class="link" href="https://www.linkedin.com/in/daren-wang-79ab6b171" target="_blank">D. Wang</a>,
                          <span class="emph">K. Lin</span>, and
                          <a class="link" href="https://voices.uchicago.edu/willett/" target="_blank">R. Willet</a><br>
                          Statistically and computationally efficient change point
localization in regression settings<br>
                         Submitted to <i> <a class="link" href="http://www.jmlr.org/" target="_blank">JMLR</a></i> (2019)
 (<a class="link" href="https://arxiv.org/abs/1906.11364" target="_blank">arxiv</a>)
        </div>
      </div>

<!-- binseg -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-binseg.png">
          <div class="right">
            <div class="research_title">
              Post-selection inference for changepoint problems
            </div>
            <p>
            Changepoint detection methods such as binary segmentation are often used in CGH analyses for copy number variation detection, but these methods lack proper downstream statistical inference. In this paper, we develop post-selection hypothesis tests for various
            changepoint detection methods and provide substantial practical guidelines based on simulation.
            </p>
          </div>
        </div>
        <div class="publication">
          <a class="link" href="http://www.stat.cmu.edu/~shyun/" target="_blank">S. Hyun</a>,
                          <span class="emph">K. Lin</span>,
                          <a class="link" href="http://www.stat.cmu.edu/people/faculty/mgsell" target="_blank">M. G'Sell</a>, and
                          <a class="link" href="http://www.stat.cmu.edu/~ryantibs/index.html" target="_blank"> R. J. Tibshirani</a><br>
                          Valid post-selection inference for segmentation methods with application to copy number variation data<br>
                         Submitted to <i> <a class="link" href="https://onlinelibrary.wiley.com/journal/15410420" target="_blank">Biometrics</a></i> (2019)
 (<a class="link" href="https://arxiv.org/abs/1812.03644" target="_blank">arxiv</a>)
 (<a class="link" href="https://github.com/robohyun66/binseginf" target="_blank">git 1</a>, <a class="link" href="https://github.com/linnylin92/selectiveModel" target="_blank">git 2</a>)
        </div>
      </div>

<!-- brainspan -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-brainspan.png">
          <div class="right">
            <div class="research_title">
              Covariance-based sample selection for heterogenous data
            </div>
            <p> Microarray samples from brain tissue is hard to collect, and also varies substantially depending on the tissue's brain region and the developmental age of its subject, hence it is hard to collect enough samples for the statistical analysis.
            In this paper, we develop a sample selection method to find additional microarray samples that are statistically similar to the samples of our desired spatio-temporal brain tissue.
            We demonstrate that after apply an existing analysis pipeline to our selected samples, we detect a higher percentage of autism risk genes.
            </p>
          </div>
        </div>
        <div class="publication">
          <span class="emph">K. Lin</span>,
                          <a class="link" href="http://magics.cs.northwestern.edu/index.html" target="_blank">H. Liu</a>, and
                          <a class="link" href="http://www.stat.cmu.edu/~roeder/" target="_blank">K. Roeder</a> <br>
                          Covariance-based sample selection for heterogenous data: Applications to gene expression and autism risk gene detection<br>
                         Submitted to <i> <a class="link" href="https://amstat.tandfonline.com/toc/uasa20/current" target="_blank">Journal of the American Statistical Association</a> (JASA) Applications and Case Studies</i> (2018)
(<a class="link" href="https://arxiv.org/abs/1812.08147" target="_blank">arxiv</a>)
(<a class="link" href="https://github.com/linnylin92/covarianceSelection" target="_blank">git</a>)
        </div>
      </div>

<div class="subtitle">Theory and Methods (Published)</div>

<!-- fused lasso -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-fused.png">
          <div class="right">
            <div class="research_title">
              Fused lasso and changepoint screening
            </div>
            <p> Changepoint estimators have statistical theory for how well they estimate the mean function and how well they estimate the changepoints, but existing theory often analyzes these properties separately. In this paper, we prove a near-optimal estimation rate for the fused lasso, which in turn directly proves a changepoint detection rate that is near the detection limit. We extend this logic to other estimators and settings.
            </p>
          </div>
        </div>
        <div class="publication">
          <span class="emph">K. Lin</span>,
                          <a class="link" href="https://jsharpna.github.io/" target="_blank">J. Sharpnack</a>,
                          <a class="link" href="http://www.stat.cmu.edu/~arinaldo/" target="_blank">A. Rinaldo</a>, and
                          <a class="link" href="http://www.stat.cmu.edu/~ryantibs/index.html" target="_blank">R. J. Tibshirani</a> <br>
                          A Sharp Error Analysis for the Fused Lasso, with Application to Approximate Changepoint Screening<br>
                          <i><a class="link" href="https://nips.cc/" target="_blank">Advances in Neural Information Processing Systems</a> (NIPS)</i> (2017)
                          (<a class="link" href="http://papers.nips.cc/paper/7264-a-sharp-error-analysis-for-the-fused-lasso-with-application-to-approximate-changepoint-screening" target="_blank">link</a>)
                      (<a class="link" href="https://arxiv.org/abs/1606.06746" target="_blank">arxiv</a>) (<a class="link" href="https://github.com/linnylin92/fused_lasso" target="_blank">git</a>)
        </div>
      </div>

<!-- cs -->
<div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-cs.png">
          <div class="right">
            <div class="research_title">
              Compressed sensing
            </div>
            <p> Many compressed sensing are developed to be as generic as possible, but have shortcomings in specialized settings where modern optimization theory can deliver a substantial boost in computational efficiency. In this paper, we develop two compressed sensing algorithms, one specialized for extremely sparse signals and another specialized for Kronecker-structed sensing matrices. We
            numerically demonstrate a near 10-times reduction in
            computation time compared to other state-of-the-art methods.
            </p>
          </div>
        </div>
        <div class="publication">
          <a class="link" href="https://vanderbei.princeton.edu/" target="_blank">R. Vanderbei</a>,
          <span class="emph">K. Lin</span>,
                          <a class="link" href="http://magics.cs.northwestern.edu/index.html" target="_blank">H. Liu</a>, and
                          <a class="link" href="http://math.mit.edu/~liewang/" target="_blank">L. Wang</a><br>
                          Revisiting compressed sensing: Exploiting the efficiency
of simplex and sparsification methods<br>
                          <i><a class="link" href="https://www.springer.com/mathematics/journal/12532" target="_blank">Mathematical Programming Computation</a></i> 8.3 (2016): 253-269.
                          (<a class="link" href="https://link.springer.com/article/10.1007/s12532-016-0105-y" target="_blank">link</a>)
                      (<a class="link" href="papers/cs.pdf" target="_blank">pdf</a>)
                      (<a class="link" href="https://github.com/linnylin92/compress_sensing" target="_blank">git</a>)
        </div>
      </div>



     <div class="subtitle">Application (Published)</div>
     <div class="block">
        <div class="abstract">
          <img class="project_img" src="images/plot-science.png">
          <div class="right">
            <div class="research_title">
              Autism risk factors within the noncoding genome
            </div>
            <p> While de novo mutations within the protein-coding portion
            of the genome have
            been thoroughly studied, these mutations in the noncoding portions
            which comprise of 98.5% of the genome
            have been less well understood. In this paper, we use a bioinformatics
            framework to analyze 1902 autism quartets via WGS and find that the
            strongest signals arose from promoters -- noncoding regions that
            control gene transcription.
            </p>
          </div>
        </div>
        <div class="publication">
           <a class="link" href="https://sanderslab.github.io/team/joonan.html" target="_blank">J. An*</a>,
           <span class="emph">K. Lin</span>*,
           <a class="link" href="https://lingxuez.github.io/" target="_blank">L. Zhu*</a>,
           <a class="link" href="https://sanderslab.github.io/team/donnawerling" target="_blank">D. M. Werling*</a>,
           S. Dong, H. Brand, H. Z. Wang, X. Zhao, G. B. Schwartz, R. L. Collins,
           B. B. Currall, C. Dastmalchi, J. Dea, C. Duhn, M. C. Gilson, L. Klei,
           L. Liang, E. Markenscoff-Papadimitriou, S. Pochareddy, N. Ahituv,
           J. D. Buxbaum, H. Coon, M. J. Daly, Y. S. Kim, G. T. Marth,
           B. M. Neale, A. R. Quinlan, J. L. Rubenstein, N. Sestan,
           M. W. State, A. J. Willsey,
           <a class="link" href="http://talkowski.mgh.harvard.edu/about/people/" target="_blank">M. E. Talkowski</a>,
           <a class="link" href="http://www.compgen.pitt.edu/" target="_blank">B. Devlin</a>,
           <a class="link" href="http://www.stat.cmu.edu/~roeder/" target="_blank">K. Roeder</a>,
           <a class="link" href="https://sanderslab.github.io/team/stephansanders" target="_blank">S. J. Sanders</a>. (*Equal contributions)
                          <br>
                          Analysis of 7,608 genomes highlights a role for promoter regions in Autism Spectrum Disorder<br>
                          <i><a class="link" href="http://science.sciencemag.org/" target="_blank">Science</a></i> 362.6420 (2018).
                          (<a class="link" href="http://science.sciencemag.org/content/362/6420" target="_blank">link</a>)
                      (<a class="link" href="http://science.sciencemag.org/content/sci/362/6420/eaat6576.full.pdf" target="_blank">pdf</a>) (<a class="link" href="https://github.com/lingxuez/WGS-Analysis" target="_blank">git</a>)
        </div>
      </div>

   <hr>


    <div class="content">
      <div class="title">Articles</div>
      <a class="anchor" id="articles"></a>

      <div class="block">
        <div class="abstract">
            <div class="application_title">
          We, the millenials: The statistical significance of political significance
            </div>
			      <div class="publication">
             <i><a class="link" href="https://www.significancemagazine.com/" target="_blank">Significance</a></i> (2017)
             (<a class="link" href="https://www.significancemagazine.com/politics/574-we-the-millennials-the-statistical-significance-of-political-significance" target="_blank">link</a>)
             </div>
          </div>
    </div>

    <hr>

    <div class="content">
      <div class="title">Talks</div>
      <a class="anchor" id="talks"></a>

      <ul>
			  <li>
			  <i>Dependency diagnostic: Visually understanding pairwise variable relationships</i> (talk).<br>
			  2018 Joint Statistical Meetings (JSM), Vancouver, Canada.
			  </li>

			  <li>
			  <i>A sharp error analysis for the fused lasso, with application to approximate changepoint screening</i> (poster).<br>
			  2017 Conference on Neural Information Processing Systems (NIPS), Long Beach, CA.
			  </li>

			  <li>
			  <i>Hypothesis testing for simulatenous variable clustering and correlation network estimation, with applications to gene coexpression networks</i> (talk).<br>
			  2017 Joint Statistical Meetings (JSM), Baltimore, MD.
			  </li>

			   <li>
			  <i>Longitudinal Gaussian graphical model for autism risk gene detection</i> (talk).<br>
			  2016 Joint Statistical Meetings (JSM), Chicago, IL.
			  </li>

			  <li>
			  <i>Longitudinal Gaussian graphical model integrating gene expression and sequencing data for autism risk gene detection</i> (talk).<br>
			  2015 American Society of Human Genetics (ASHG), Baltimore, MD.
			  </li>

			  <li>
			  <i>Optimization for compressed sensing: New insights and alternatives</i> (talk).<br>
			  2014 Modeling and Optimization: Theory and Applications, Bethlehem, PA.
			  </li>
      </ul>
    </div>

    <hr>

    <div class="content">
      <div class="title">Experience</div>
      <a class="anchor" id="experience"></a>

      <div class="subtitle">Honors and Awards</div>
      <ul>
             <li>
        <i>Honorable mention in student paper competition</i><br>
        (For article "Dependency diagnostic: Visually understanding pairwise variable relationships")<br>
        ASA section: Statistical Computing and Statistical Graphics, January 2018
        </li>

        <li>
        <i>Winner of Statistical excellence for early-career writing</i><br>
        (For article "We, the millenials: The statistical significance of political significance")<br>
        Significance magazine in partnership with Young Statisticians Section of Royal Statistical Society, June 2017
        </li>

        <li>
        <i>Teaching assistant award recipient</i><br>
        (For "Statistical Computing" in Fall 2016)<br>
        Carnegie Mellon University, May 2017
        </li>

         <li>
        <i>Award recipient of Kenneth H. Condit Prize</i><br>
        (For excellence in service to department)<br>
        Princeton University, May 2014
        </li>

      </ul>

    <div class="subtitle">Teaching Experience</div>

      <ul>
        <li>(2019 Spring) 36-490 Undergraduate Research (Data Science Initiative Project Fellow under <a class="link" href="http://www.stat.cmu.edu/~rnugent/PUBLIC/" target="_blank">R. Nugent</a> and <a class="link" href="http://www.stat.cmu.edu/~pfreeman/" target="_blank">P. Freeman</a>)</li>
			   <li>(2018 Summer) 36-350 Statistical Computing (Instructor)</li>
			   <li>(2018 Spring) 36-350 Statistical Computing (Assistant Instructor with <a class="link" href="http://www.stat.cmu.edu/~ryantibs/index.html" target="_blank">R. J. Tibshirani</a>)</li>
			   <li>(2017 Fall) 36-350 Statistical Computing (TA under <a class="link" href="http://www.stat.cmu.edu/~pfreeman/" target="_blank">P. Freeman</a>)</li>
			   <li>(2015 Fall, 2016 Fall) 36-350 Statistical Computing (TA under <a class="link" href="http://www.stat.cmu.edu/~ryantibs/index.html" target="_blank">R. J. Tibshirani</a>)</li>
			    <li>(2015 Spring) 36-217 Probability Theory and Random Processes (TA under <a class="link" href="http://www.stat.cmu.edu/~arinaldo/" target="_blank">A. Rinaldo</a>)</li>
			    <li>(2014 Fall) 46-921 Financial Data Analysis I and 46923 Financial Data Analysis II (TA under <a class="link" href="http://www.stat.cmu.edu/~cschafer/" target="_blank">C. Schafer</a>)</li>
			    <li>(2014 Spring, 2013 Spring, 2012 Spring) ORF 350 Analysis of Big Data (Course designer under <a class="link" href="http://magics.cs.northwestern.edu/index.html" target="_blank">H. Liu</a>)</li>
      </ul>

      <div class="subtitle">Professional Service</div>

      <ul>
        <li>
          Reviewer of <a class="link" href="https://projecteuclid.org/euclid.aos" target="_blank">Annals of Statistics</a>, <a class="link" href="https://www.tandfonline.com/toc/uasa20/current" target="_blank">Journal of the American Statistical Association</a>, <a class="link" href="https://academic.oup.com/biomet" target="_blank">Biometrika</a>, <a class="link" href="https://projecteuclid.org/euclid.ejs" target="_blank">Electronic Journal of Statistics</a>, <a class="link" href="http://www3.stat.sinica.edu.tw/statistica/" target="_blank">Statistical Sinica</a>, <a class="link" href="https://ieeexplore.ieee.org/xpl/RecentIssue.jsp?punumber=6488902" target="_blank">IEEE Transactions on Network Science and Engineering</a>
          </li>
          <li>
          Founding member of <a class="link" href="http://www.stat.cmu.edu/help-network/" target="_blank">CMU Statistics department help network</a>
          </li>
      </ul>



    </div>
  <div style="margin-top: 10px; text-align: center; font-size: 14px;">
  Copyright &copy; 2015 - 2018, Kevin Lin. All rights reserved.<br>
  Last Updated: August 10, 2018.
  </div>

</body>
</html>
